// @clerk/localizations/dist/fr-FR.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/fr-FR.ts
var fr_FR_exports = {};
__export(fr_FR_exports, {
  frFR: () => frFR
});
module.exports = __toCommonJS(fr_FR_exports);
var frFR = {
  locale: "fr-FR",
  backButton: "Retour",
  badge__default: "D\xE9faut",
  badge__otherImpersonatorDevice: "Autre dispositif d'imitation",
  badge__primary: "Principal",
  badge__requiresAction: "N\xE9cessite une action",
  badge__thisDevice: "Cet appareil",
  badge__unverified: "Non v\xE9rifi\xE9",
  badge__userDevice: "Appareil utilisateur",
  badge__you: "Vous",
  createOrganization: {
    formButtonSubmit: "Cr\xE9er l\u2019organisation",
    invitePage: {
      formButtonReset: "Passer"
    },
    title: "Cr\xE9er une organisation"
  },
  dates: {
    lastDay: "Hier \xE0 {{ date | timeString('fr-FR') }}",
    next6Days: "{{ date | weekday('fr-FR','long') }} \xE0 {{ date | timeString('fr-FR') }}",
    nextDay: "Demain \xE0 {{ date | timeString('fr-FR') }}",
    numeric: "{{ date | numeric('fr-FR') }}",
    previous6Days: "{{ date | weekday('fr-FR','long') }} dernier \xE0 {{ date | timeString('fr-FR') }}",
    sameDay: "Aujourd'hui \xE0 {{ date | timeString('fr-FR') }}"
  },
  dividerText: "ou",
  footerActionLink__useAnotherMethod: "Utiliser une autre m\xE9thode",
  footerPageLink__help: "Aider",
  footerPageLink__privacy: "Intimit\xE9",
  footerPageLink__terms: "Conditions",
  formButtonPrimary: "Continuer",
  formButtonPrimary__verify: "Verify",
  formFieldAction__forgotPassword: "Mot de passe oubli\xE9?",
  formFieldError__matchingPasswords: "Les mots de passe correspondent.",
  formFieldError__notMatchingPasswords: "Les mots de passe ne correspondent pas.",
  formFieldError__verificationLinkExpired: "The verification link expired. Please request a new link.",
  formFieldHintText__optional: "Optionnel",
  formFieldHintText__slug: "A slug is a human-readable ID that must be unique. It\u2019s often used in URLs.",
  formFieldInputPlaceholder__backupCode: "",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Delete account",
  formFieldInputPlaceholder__emailAddress: "",
  formFieldInputPlaceholder__emailAddress_username: "",
  formFieldInputPlaceholder__emailAddresses: "Saisissez ou collez une ou plusieurs adresses e-mail, s\xE9par\xE9es par des espaces ou des virgules",
  formFieldInputPlaceholder__firstName: "",
  formFieldInputPlaceholder__lastName: "",
  formFieldInputPlaceholder__organizationDomain: "",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "",
  formFieldInputPlaceholder__organizationName: "",
  formFieldInputPlaceholder__organizationSlug: "",
  formFieldInputPlaceholder__password: "",
  formFieldInputPlaceholder__phoneNumber: "",
  formFieldInputPlaceholder__username: "",
  formFieldLabel__automaticInvitations: "Enable automatic invitations for this domain",
  formFieldLabel__backupCode: "Code de r\xE9cup\xE9ration",
  formFieldLabel__confirmDeletion: "Confirmation",
  formFieldLabel__confirmPassword: "Confirmez le mot de passe",
  formFieldLabel__currentPassword: "Mot de passe actuel",
  formFieldLabel__emailAddress: "Adresse e-mail",
  formFieldLabel__emailAddress_username: "Adresse email ou nom d'utilisateur",
  formFieldLabel__emailAddresses: "Adresses e-mail",
  formFieldLabel__firstName: "Pr\xE9nom",
  formFieldLabel__lastName: "Nom de famille",
  formFieldLabel__newPassword: "Nouveau mot de passe",
  formFieldLabel__organizationDomain: "Domain",
  formFieldLabel__organizationDomainDeletePending: "Delete pending invitations and suggestions",
  formFieldLabel__organizationDomainEmailAddress: "Verification email address",
  formFieldLabel__organizationDomainEmailAddressDescription: "Enter an email address under this domain to receive a code and verify this domain.",
  formFieldLabel__organizationName: "Nom de l'organisation",
  formFieldLabel__organizationSlug: "Slug URL",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "Mot de passe",
  formFieldLabel__phoneNumber: "Num\xE9ro de t\xE9l\xE9phone",
  formFieldLabel__role: "R\xF4le",
  formFieldLabel__signOutOfOtherSessions: "Se d\xE9connecter de tous les autres appareils",
  formFieldLabel__username: "Nom d'utilisateur",
  impersonationFab: {
    action__signOut: "D\xE9connexion",
    title: "Connect\xE9 en tant que {{identifier}}"
  },
  maintenanceMode: void 0,
  membershipRole__admin: "Administrateur",
  membershipRole__basicMember: "Membre",
  membershipRole__guestMember: "Invit\xE9",
  organizationList: {
    action__createOrganization: "Create organization",
    action__invitationAccept: "Join",
    action__suggestionsAccept: "Request to join",
    createOrganization: "Create Organization",
    invitationAcceptedLabel: "Joined",
    subtitle: "to continue to {{applicationName}}",
    suggestionsAcceptedLabel: "Pending approval",
    title: "Choose an account",
    titleWithoutPersonal: "Choose an organization"
  },
  organizationProfile: {
    badge__automaticInvitation: "Automatic invitations",
    badge__automaticSuggestion: "Automatic suggestions",
    badge__manualInvitation: "No automatic enrollment",
    badge__unverified: "Unverified",
    createDomainPage: {
      subtitle: "Add the domain to verify. Users with email addresses at this domain can join the organization automatically or request to join.",
      title: "Add domain"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Les invitations suivantes n\u2019ont pas pu \xEAtre envoy\xE9es. Veuillez r\xE9gler ce probl\xE8me et r\xE9essayer:",
      formButtonPrimary__continue: "Envoyer des invitations",
      selectDropdown__role: "Select role",
      subtitle: "Inviter des membres \xE0 rejoindre l\u2019organisation",
      successMessage: "Les invitations ont \xE9t\xE9 envoy\xE9es.",
      title: "Inviter des membres"
    },
    membersPage: {
      action__invite: "Inviter",
      activeMembersTab: {
        menuAction__remove: "Supprimer",
        tableHeader__actions: "",
        tableHeader__joined: "Rejoint",
        tableHeader__role: "R\xF4le",
        tableHeader__user: "Utilisateur"
      },
      detailsTitle__emptyRow: "Aucun membre",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invite users by connecting an email domain with your organization. Anyone who signs up with a matching email domain will be able to join the organization anytime.",
          headerTitle: "Automatic invitations",
          primaryButton: "Manage verified domains"
        },
        table__emptyRow: "No invitations to display"
      },
      invitedMembersTab: {
        menuAction__revoke: "R\xE9voquer l'invitation",
        tableHeader__invited: "Invit\xE9"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Users who sign up with a matching email domain, will be able to see a suggestion to request to join your organization.",
          headerTitle: "Automatic suggestions",
          primaryButton: "Manage verified domains"
        },
        menuAction__approve: "Approve",
        menuAction__reject: "Reject",
        tableHeader__requested: "Requested access",
        table__emptyRow: "No requests to display"
      },
      start: {
        headerTitle__invitations: "Invitations",
        headerTitle__members: "Members",
        headerTitle__requests: "Requests"
      }
    },
    navbar: {
      description: "Manage your organization.",
      general: "General",
      members: "Members",
      title: "Organization"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: "Saisissez {{organizationName}} ci-dessous pour continuer.",
          messageLine1: "\xCAtes-vous s\xFBr(e) de vouloir supprimer cette organisation ?",
          messageLine2: "Cette action est d\xE9finitive et irr\xE9versible.",
          successMessage: "Vous avez supprim\xE9 l'organisation.",
          title: "Supprimer l'organisation"
        },
        leaveOrganization: {
          actionDescription: "Saisissez {{organizationName}} ci-dessous pour continuer.",
          messageLine1: "\xCAtes-vous s\xFBr de vouloir quitter cette organisation ? Vous perdrez l'acc\xE8s \xE0 cette organisation et \xE0 ses applications.",
          messageLine2: "Cette action est permanente et irr\xE9versible.",
          successMessage: "Vous avez quitt\xE9 l'organisation.",
          title: "Quitter l'organisation"
        },
        title: "Danger"
      },
      domainSection: {
        menuAction__manage: "Manage",
        menuAction__remove: "Delete",
        menuAction__verify: "Verify",
        primaryButton: "Add domain",
        subtitle: "Allow users to join the organization automatically or request to join based on a verified email domain.",
        title: "Verified domains"
      },
      successMessage: "L'organisation a \xE9t\xE9 mise \xE0 jour.",
      title: "Profil de l\u2019organisation"
    },
    removeDomainPage: {
      messageLine1: "The email domain {{domain}} will be removed.",
      messageLine2: "Users won\u2019t be able to join the organization automatically after this.",
      successMessage: "{{domain}} has been removed.",
      title: "Remove domain"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "Membres",
      profileSection: {
        primaryButton: "",
        title: "Organization Profile",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Removing this domain will affect invited users.",
        removeDomainActionLabel__remove: "Remove domain",
        removeDomainSubtitle: "Remove this domain from your verified domains",
        removeDomainTitle: "Remove domain"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Users are automatically invited to join the organization when they sign-up and can join anytime.",
        automaticInvitationOption__label: "Automatic invitations",
        automaticSuggestionOption__description: "Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.",
        automaticSuggestionOption__label: "Automatic suggestions",
        calloutInfoLabel: "Changing the enrollment mode will only affect new users.",
        calloutInvitationCountLabel: "Pending invitations sent to users: {{count}}",
        calloutSuggestionCountLabel: "Pending suggestions sent to users: {{count}}",
        manualInvitationOption__description: "Users can only be invited manually to the organization.",
        manualInvitationOption__label: "No automatic enrollment",
        subtitle: "Choose how users from this domain can join the organization."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Enrollment options"
      },
      subtitle: "The domain {{domain}} is now verified. Continue by selecting enrollment mode.",
      title: "Update {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Enter the verification code sent to your email address",
      formTitle: "Verification code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "The domain {{domainName}} needs to be verified via email.",
      subtitleVerificationCodeScreen: "A verification code was sent to {{emailAddress}}. Enter the code to continue.",
      title: "Verify domain"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "Cr\xE9er une organisation",
    action__invitationAccept: "Join",
    action__manageOrganization: "G\xE9rer l'organisation",
    action__suggestionsAccept: "Request to join",
    notSelected: "Aucune organisation s\xE9lectionn\xE9e",
    personalWorkspace: "Espace de travail personnel",
    suggestionsAcceptedLabel: "Pending approval"
  },
  paginationButton__next: "Prochain",
  paginationButton__previous: "Pr\xE9c\xE9dent",
  paginationRowText__displaying: "Affichage",
  paginationRowText__of: "de",
  signIn: {
    accountSwitcher: {
      action__addAccount: "Add account",
      action__signOutAll: "Sign out of all accounts",
      subtitle: "Select the account with which you wish to continue.",
      title: "Choose an account"
    },
    alternativeMethods: {
      actionLink: "Obtenir de l'aide",
      actionText: "Don\u2019t have any of these?",
      blockButton__backupCode: "Utiliser un code de r\xE9cup\xE9ration",
      blockButton__emailCode: "Envoyer le code \xE0 {{identifier}}",
      blockButton__emailLink: "Envoyer le lien \xE0 {{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "Connectez-vous avec votre mot de passe",
      blockButton__phoneCode: "Envoyer le code \xE0 {{identifier}}",
      blockButton__totp: "Utilisez votre application d'authentification",
      getHelp: {
        blockButton__emailSupport: "Assistance par e-mail",
        content: "Si vous rencontrez des difficult\xE9s pour vous connecter \xE0 votre compte, envoyez-nous un e-mail et nous travaillerons avec vous pour r\xE9tablir l'acc\xE8s d\xE8s que possible.",
        title: "Obtenir de l'aide"
      },
      subtitle: "Facing issues? You can use any of these methods to sign in.",
      title: "Utiliser une autre m\xE9thode"
    },
    backupCodeMfa: {
      subtitle: "pour continuer vers {{applicationName}}",
      title: "Entrez un code de r\xE9cup\xE9ration"
    },
    emailCode: {
      formTitle: "Le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre messagerie"
    },
    emailLink: {
      expired: {
        subtitle: "Retournez \xE0 l'onglet d'origine pour continuer",
        title: "Ce lien de v\xE9rification a expir\xE9"
      },
      failed: {
        subtitle: "Retourner \xE0 l'onglet original pour continuer",
        title: "Ce lien de v\xE9rification n'est pas valide"
      },
      formSubtitle: "Utilisez le lien de v\xE9rification envoy\xE9 par e-mail",
      formTitle: "lien de v\xE9rification",
      loading: {
        subtitle: "Vous allez bient\xF4t \xEAtre redirig\xE9",
        title: "Signing in..."
      },
      resendButton: "Renvoyer le lien",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre messagerie",
      unusedTab: {
        title: "Vous pouvez fermer cet onglet"
      },
      verified: {
        subtitle: "Vous serez bient\xF4t redirig\xE9",
        title: "Connexion r\xE9ussie"
      },
      verifiedSwitchTab: {
        subtitle: "Revenir \xE0 l'onglet d'origine pour continuer",
        subtitleNewTab: "Revenez \xE0 l'onglet nouvellement ouvert pour continuer",
        titleNewTab: "Connect\xE9 sur un autre onglet"
      }
    },
    forgotPassword: {
      formTitle: "Code de r\xE9initialisation du mot de passe",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "to reset your password",
      subtitle_email: "First, enter the code sent to your email ID",
      subtitle_phone: "First, enter the code sent to your phone",
      title: "Reset password"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "R\xE9initialiser votre mot de passe",
      label__alternativeMethods: "Ou connectez-vous avec une autre m\xE9thode.",
      title: "Mot de passe oubli\xE9 ?"
    },
    noAvailableMethods: {
      message: "Impossible de poursuivre la connexion. Aucun facteur d'authentification n'est disponible.",
      subtitle: "Une erreur s'est produite",
      title: "Impossible de se connecter"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "Utiliser une autre m\xE9thode",
      subtitle: "pour continuer \xE0 {{applicationName}}",
      title: "Tapez votre mot de passe"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "Code de v\xE9rification",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "pour continuer sur {{applicationName}}",
      title: "V\xE9rifiez votre t\xE9l\xE9phone"
    },
    phoneCodeMfa: {
      formTitle: "Code de v\xE9rification",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "",
      title: "V\xE9rifiez votre t\xE9l\xE9phone"
    },
    resetPassword: {
      formButtonPrimary: "R\xE9initialiser",
      requiredMessage: "For security reasons, it is required to reset your password.",
      successMessage: "Votre mot de passe a \xE9t\xE9 modifi\xE9 avec succ\xE8s. Nous vous reconnectons, veuillez patienter un instant.",
      title: "R\xE9initialiser le mot de passe"
    },
    resetPasswordMfa: {
      detailsLabel: "Nous devons v\xE9rifier votre identit\xE9 avant de r\xE9initialiser votre mot de passe."
    },
    start: {
      actionLink: "S'inscrire",
      actionLink__use_email: "Utiliser e-mail",
      actionLink__use_email_username: "Utiliser l'e-mail ou le nom d'utilisateur",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "Utiliser t\xE9l\xE9phone",
      actionLink__use_username: "Utiliser le nom d'utilisateur",
      actionText: "Vous n'avez pas encore de compte ?",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "S'identifier"
    },
    totpMfa: {
      formTitle: "Le code de v\xE9rification",
      subtitle: "",
      title: "V\xE9rification en deux \xE9tapes"
    }
  },
  signInEnterPasswordTitle: "Tapez votre mot de passe",
  signUp: {
    continue: {
      actionLink: "S'identifier",
      actionText: "Vous avez d\xE9j\xE0 un compte ?",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "Remplir les champs manquants"
    },
    emailCode: {
      formSubtitle: "Entrez le code de v\xE9rification envoy\xE9 \xE0 votre adresse e-mail",
      formTitle: "Le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "pour continuer \xE0 {{applicationName}}",
      title: "V\xE9rifiez votre e-mail"
    },
    emailLink: {
      formSubtitle: "Utilisez le lien de v\xE9rification envoy\xE9 \xE0 votre adresse e-mail",
      formTitle: "lien de v\xE9rification",
      loading: {
        title: "Cr\xE9ation de votre compte..."
      },
      resendButton: "Renvoyer le lien",
      subtitle: "pour continuer \xE0 {{applicationName}}",
      title: "V\xE9rifiez votre e-mail",
      verified: {
        title: "Compte cr\xE9\xE9"
      },
      verifiedSwitchTab: {
        subtitle: "Revenez \xE0 l'onglet nouvellement ouvert pour continuer",
        subtitleNewTab: "Revenir \xE0 l'onglet pr\xE9c\xE9dent pour continuer",
        title: "Courriel v\xE9rifi\xE9 avec succ\xE8s"
      }
    },
    phoneCode: {
      formSubtitle: "Entrez le code de v\xE9rification envoy\xE9 \xE0 votre num\xE9ro de t\xE9l\xE9phone",
      formTitle: "Le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "pour continuer \xE0 {{applicationName}}",
      title: "V\xE9rifiez votre t\xE9l\xE9phone"
    },
    start: {
      actionLink: "S'identifier",
      actionText: "Vous avez d\xE9j\xE0 un compte ?",
      subtitle: "pour continuer \xE0 {{applicationName}}",
      title: "Cr\xE9ez votre compte"
    }
  },
  socialButtonsBlockButton: "Continuer avec {{provider|titleize}}",
  unstable__errors: {
    captcha_invalid: "Inscription \xE9chou\xE9e en raison de validations de s\xE9curit\xE9 incorrectes. Veuillez rafra\xEEchir la page pour r\xE9essayer ou contacter le support pour obtenir de l'aide.",
    captcha_unavailable: "Sign up unsuccessful due to failed bot validation. Please refresh the page to try again or reach out to support for more assistance.",
    form_code_incorrect: "Code incorrect",
    form_identifier_exists: "Cet identifiant existe d\xE9j\xE0. Veuillez en renseigner un diff\xE9rent.",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "Identifiant introuvable",
    form_param_format_invalid: "Le format est invalide",
    form_param_format_invalid__email_address: "L'adresse e-mail doit \xEAtre une adresse e-mail valide.",
    form_param_format_invalid__phone_number: "Phone number must be in a valid international format",
    form_param_max_length_exceeded__first_name: "First name should not exceed 256 characters.",
    form_param_max_length_exceeded__last_name: "Last name should not exceed 256 characters.",
    form_param_max_length_exceeded__name: "Name should not exceed 256 characters.",
    form_param_nil: "",
    form_password_incorrect: "Mot de passe incorrect",
    form_password_length_too_short: "Votre mot de passe est trop court.",
    form_password_not_strong_enough: "Votre mot de passe n'est pas assez fort.",
    form_password_pwned: "Ce mot de passe a \xE9t\xE9 compromis et ne peut pas \xEAtre utilis\xE9. Veuillez essayer un autre mot de passe \xE0 la place.",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "Votre mot de passe a d\xE9pass\xE9 le nombre maximum d'octets autoris\xE9s. Veuillez le raccourcir ou supprimer certains caract\xE8res sp\xE9ciaux.",
    form_password_validation_failed: "Mot de passe incorrect",
    form_username_invalid_character: "L'identifiant contient des caract\xE8res invalide.",
    form_username_invalid_length: "Le nombre de caract\xE8res de l'identifiant est invalide.",
    identification_deletion_failed: "Vous ne pouvez pas supprimer votre derni\xE8re identification.",
    not_allowed_access: "Acc\xE8s non autoris\xE9",
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "moins de {{length}} caract\xE8res",
      minimumLength: "{{length}} caract\xE8res ou plus",
      requireLowercase: "une lettre minuscule",
      requireNumbers: "un chiffre",
      requireSpecialCharacter: "un caract\xE8re sp\xE9cial",
      requireUppercase: "une lettre majuscule",
      sentencePrefix: "Votre mot de passe doit contenir"
    },
    phone_number_exists: "Ce num\xE9ro de t\xE9l\xE9phone est d\xE9j\xE0 utilis\xE9. Veuillez essayer un autre.",
    zxcvbn: {
      couldBeStronger: "Votre mot de passe fonctionne mais pourrait \xEAtre plus s\xFBr. Essayez d'ajouter des caract\xE8res.",
      goodPassword: "Bien jou\xE9. C'est un excellent mot de passe.",
      notEnough: "Votre mot de passe n'est pas assez fort.",
      suggestions: {
        allUppercase: "Mettez quelques lettres en majuscules, mais pas toutes.",
        anotherWord: "Ajoutez des mots moins courants.",
        associatedYears: "\xC9vitez les ann\xE9es qui vous sont associ\xE9es. (ex: date de naissance)",
        capitalization: "Capitalisez mais pas seulement la premi\xE8re lettre.",
        dates: "\xC9vitez les dates et les ann\xE9es qui vous sont associ\xE9es. (ex: date ou ann\xE9e de naissance)",
        l33t: "\xC9vitez les substitutions de lettres pr\xE9visibles comme '@' pour 'a'.",
        longerKeyboardPattern: "Utilisez des motifs de clavier plus longs et changez de sens de frappe plusieurs fois.",
        noNeed: "Vous pouvez cr\xE9er des mots de passe forts sans utiliser de symboles, de chiffres ou de lettres majuscules.",
        pwned: "Si vous utilisez ce mot de passe ailleurs, vous devriez le modifier.",
        recentYears: "\xC9vitez les derni\xE8res ann\xE9es.",
        repeated: "\xC9vitez les mots et les caract\xE8res r\xE9p\xE9t\xE9s.",
        reverseWords: "\xC9vitez les orthographes invers\xE9es des mots courants",
        sequences: "\xC9vitez les s\xE9quences de caract\xE8res courantes.",
        useWords: "Utilisez plusieurs mots, mais \xE9vitez les phrases courantes."
      },
      warnings: {
        common: "Ce mot de passe est couramment utilis\xE9.",
        commonNames: "Les noms communs et les noms de famille sont faciles \xE0 deviner.",
        dates: "Les dates sont faciles \xE0 deviner.",
        extendedRepeat: "Les caract\xE8res r\xE9p\xE9t\xE9s comme 'abcabcabc' sont faciles \xE0 deviner.",
        keyPattern: "Des motifs de clavier courts sont faciles \xE0 deviner.",
        namesByThemselves: "Des noms ou des pr\xE9noms simples sont faciles \xE0 deviner.",
        pwned: "Votre mot de passe a \xE9t\xE9 divulgu\xE9 suite \xE0 une violation de donn\xE9es sur Internet.",
        recentYears: "Les ann\xE9es r\xE9centes sont faciles \xE0 deviner.",
        sequences: "Des s\xE9quences de caract\xE8res communs comme 'abc' sont faciles \xE0 deviner.",
        similarToCommon: "Ce mot de passe est similaire \xE0 un mot de passe couramment utilis\xE9.",
        simpleRepeat: "Les caract\xE8res r\xE9p\xE9t\xE9s comme 'aaa' sont faciles \xE0 deviner.",
        straightRow: "Les lignes droites de touches de votre clavier sont faciles \xE0 deviner.",
        topHundred: "Ce mot de passe est fr\xE9quemment utilis\xE9.",
        topTen: "Ce mot de passe est tr\xE8s utilis\xE9.",
        userInputs: "Le mot de passe ne doit pas comporter de donn\xE9es personnelles ou li\xE9es au site.",
        wordByItself: "Les mots simples sont faciles \xE0 deviner."
      }
    }
  },
  userButton: {
    action__addAccount: "Ajouter un compte",
    action__manageAccount: "G\xE9rer son compte",
    action__signOut: "D\xE9connexion",
    action__signOutAll: "Se d\xE9connecter de tous les comptes"
  },
  userProfile: {
    backupCodePage: {
      actionLabel__copied: "Copi\xE9 !",
      actionLabel__copy: "Copier tous les codes",
      actionLabel__download: "T\xE9l\xE9charger en .txt",
      actionLabel__print: "Imprimer",
      infoText1: "Les codes de r\xE9cup\xE9ration seront activ\xE9s pour ce compte.",
      infoText2: "Gardez les codes de r\xE9cup\xE9ration secrets et stockez-les en toute s\xE9curit\xE9. Vous pouvez r\xE9g\xE9n\xE9rer les codes de r\xE9cup\xE9ration si vous pensez qu'ils ont \xE9t\xE9 compromis.",
      subtitle__codelist: "Conservez-les en toute s\xE9curit\xE9 et gardez-les secrets.",
      successMessage: "Les codes de r\xE9cup\xE9ration sont maintenant activ\xE9s. Vous pouvez utiliser l'un d'entre eux pour vous connecter \xE0 votre compte, si vous perdez l'acc\xE8s \xE0 votre dispositif d'authentification. Chaque code ne peut \xEAtre utilis\xE9 qu'une seule fois.",
      successSubtitle: "Vous pouvez utiliser l'un d'entre eux pour vous connecter \xE0 votre compte, si vous perdez l'acc\xE8s \xE0 votre dispositif d'authentification.",
      title: "Ajouter la v\xE9rification du code de r\xE9cup\xE9ration",
      title__codelist: "Codes de r\xE9cup\xE9ration"
    },
    connectedAccountPage: {
      formHint: "S\xE9lectionnez un fournisseur pour connecter votre compte.",
      formHint__noAccounts: "Aucun fournisseur de compte externe n'est disponible.",
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus utiliser ce compte connect\xE9 et toutes les fonctionnalit\xE9s d\xE9pendantes ne fonctionneront plus.",
        successMessage: "{{connectedAccount}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer le compte connect\xE9"
      },
      socialButtonsBlockButton: "Connecter {{provider|titleize}} compte",
      successMessage: "Le fournisseur a \xE9t\xE9 ajout\xE9 \xE0 votre compte",
      title: "Ajouter un compte connect\xE9"
    },
    deletePage: {
      actionDescription: 'Saisissez "Supprimer le compte" ci-dessous pour continuer.',
      confirm: "Supprimer le compte",
      messageLine1: "\xCAtes-vous s\xFBr(e) de vouloir supprimer votre compte ?",
      messageLine2: "Cette action est d\xE9finitive et irr\xE9versible.",
      title: "Supprimer le compte"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "Un e-mail contenant un code de v\xE9rification sera envoy\xE9 \xE0 cette adresse e-mail.",
        formSubtitle: "Saisissez le code de v\xE9rification envoy\xE9 \xE0 {{identifier}}",
        formTitle: "Le code de v\xE9rification",
        resendButton: "Renvoyer le lien",
        successMessage: "L'e-mail {{identifier}} a \xE9t\xE9 v\xE9rifi\xE9 et ajout\xE9 \xE0 votre compte."
      },
      emailLink: {
        formHint: "Un e-mail contenant un lien de v\xE9rification sera envoy\xE9 \xE0 cette adresse e-mail.",
        formSubtitle: "Cliquez sur le lien de v\xE9rification dans l'e-mail envoy\xE9 \xE0 {{identifier}}",
        formTitle: "lien de v\xE9rification",
        resendButton: "Renvoyer le lien",
        successMessage: "L'e-mail {{identifier}} a \xE9t\xE9 v\xE9rifi\xE9 et ajout\xE9 \xE0 votre compte."
      },
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus vous connecter avec cette adresse e-mail.",
        successMessage: "{{emailAddress}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer l'adresse e-mail"
      },
      title: "Ajouter une adresse e-mail",
      verifyTitle: "Verify email address"
    },
    formButtonPrimary__add: "Add",
    formButtonPrimary__continue: "Continuer",
    formButtonPrimary__finish: "Retour",
    formButtonPrimary__remove: "Remove",
    formButtonPrimary__save: "Save",
    formButtonReset: "Annuler",
    mfaPage: {
      formHint: "S\xE9lectionnez une m\xE9thode \xE0 ajouter.",
      title: "Ajouter la v\xE9rification en deux \xE9tapes"
    },
    mfaPhoneCodePage: {
      backButton: "Use existing number",
      primaryButton__addPhoneNumber: "Ajouter un num\xE9ro de t\xE9l\xE9phone",
      removeResource: {
        messageLine1: "{{identifier}} ne recevra plus de codes de validation lors de la connexion.",
        messageLine2: "Votre compte sera moins s\xE9curis\xE9. Souhaitez-vous continuer ?",
        successMessage: "La v\xE9rification en deux \xE9tapes du code SMS a \xE9t\xE9 supprim\xE9e pour {{mfaPhoneCode}}",
        title: "Supprimer la v\xE9rification en deux \xE9tapes"
      },
      subtitle__availablePhoneNumbers: "S\xE9lectionnez un num\xE9ro de t\xE9l\xE9phone pour vous inscrire \xE0 la v\xE9rification en deux \xE9tapes du code SMS.",
      subtitle__unavailablePhoneNumbers: "Il n'y a pas de num\xE9ros de t\xE9l\xE9phone disponibles pour s'inscrire \xE0 la v\xE9rification en deux \xE9tapes du code SMS.",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS code verification enabled",
      title: "Ajouter la v\xE9rification du code SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Scannez le code QR \xE0 la place",
        buttonUnableToScan__nonPrimary: "Vous ne pouvez pas scanner le code QR ?",
        infoText__ableToScan: "Configurez une nouvelle m\xE9thode de connexion dans votre application d'authentification et scannez le code QR suivant pour le lier \xE0 votre compte.",
        infoText__unableToScan: "Configurez une nouvelle m\xE9thode de connexion dans votre authentificateur et entrez la cl\xE9 fournie ci-dessous.",
        inputLabel__unableToScan1: "Assurez-vous que les mots de passe bas\xE9s sur le temps ou \xE0 usage unique sont activ\xE9s, puis terminez la liaison de votre compte.",
        inputLabel__unableToScan2: "Alternativement, si votre authentificateur prend en charge les URI TOTP, vous pouvez \xE9galement copier l'URI complet."
      },
      removeResource: {
        messageLine1: "Les codes de v\xE9rification de cet authentificateur ne seront plus requis lors de la connexion.",
        messageLine2: "Votre compte sera moins s\xE9curis\xE9. Souhaitez-vous continuer ?",
        successMessage: "La v\xE9rification en deux \xE9tapes via l'application d'authentification a \xE9t\xE9 supprim\xE9e.",
        title: "Supprimer la v\xE9rification en deux \xE9tapes"
      },
      successMessage: "La v\xE9rification en deux \xE9tapes est maintenant activ\xE9e. Lors de la connexion, vous devrez saisir un code de v\xE9rification de cet authentificateur comme \xE9tape suppl\xE9mentaire.",
      title: "Ajouter une application d'authentification",
      verifySubtitle: "Entrez le code de v\xE9rification g\xE9n\xE9r\xE9 par votre application d'authentification",
      verifyTitle: "Le code de v\xE9rification"
    },
    mobileButton__menu: "Menu",
    navbar: {
      account: "Profile",
      description: "Manage your account info.",
      security: "Security",
      title: "Account"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "It is recommended to sign out of all other devices which may have used your old password.",
      readonly: "Your password can currently not be edited because you can sign in only via the enterprise connection.",
      successMessage__set: "Votre mot de passe a \xE9t\xE9 mis \xE0 jour.",
      successMessage__signOutOfOtherSessions: "Tous les autres appareils ont \xE9t\xE9 d\xE9connect\xE9s.",
      successMessage__update: "Votre mot de passe a \xE9t\xE9 mis \xE0 jour.",
      title__set: "Mettre \xE0 jour le mot de passe",
      title__update: "Changer le mot de passe"
    },
    phoneNumberPage: {
      infoText: "Un message texte contenant un lien de v\xE9rification sera envoy\xE9 \xE0 ce num\xE9ro de t\xE9l\xE9phone.",
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus vous connecter avec ce num\xE9ro de t\xE9l\xE9phone.",
        successMessage: "{{phoneNumber}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer le num\xE9ro de t\xE9l\xE9phone"
      },
      successMessage: "{{identifier}} a \xE9t\xE9 v\xE9rifi\xE9 et ajout\xE9 \xE0 votre compte.",
      title: "Ajouter un num\xE9ro de t\xE9l\xE9phone",
      verifySubtitle: "Enter the verification code sent to {{identifier}}",
      verifyTitle: "Verify phone number"
    },
    profilePage: {
      fileDropAreaHint: "T\xE9l\xE9chargez une image JPG, PNG, GIF ou WEBP inf\xE9rieure \xE0 10 Mo",
      imageFormDestructiveActionSubtitle: "Supprimer l'image",
      imageFormSubtitle: "T\xE9l\xE9charger une image",
      imageFormTitle: "Photo de profil",
      readonly: "Les informations de votre profil ont \xE9t\xE9 fournies par la connexion d'entreprise et ne peuvent pas \xEAtre modifi\xE9es.",
      successMessage: "Votre profil a \xE9t\xE9 mis a jour.",
      title: "Mettre \xE0 jour le profil"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Se d\xE9connecter de l'appareil",
        title: "Appareils actifs"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "R\xE9essayer",
        actionLabel__reauthorize: "Authorize now",
        destructiveActionTitle: "Retirer",
        primaryButton: "Connecter le compte",
        subtitle__reauthorize: "The required scopes have been updated, and you may be experiencing limited functionality. Please re-authorize this application to avoid any issues",
        title: "Comptes connect\xE9s"
      },
      dangerSection: {
        deleteAccountButton: "Supprimer le compte",
        title: "Danger"
      },
      emailAddressesSection: {
        destructiveAction: "Supprimer l'adresse e-mail",
        detailsAction__nonPrimary: "D\xE9finir comme principale",
        detailsAction__primary: "Compl\xE9ter la v\xE9rification",
        detailsAction__unverified: "Compl\xE9ter la v\xE9rification",
        primaryButton: "Ajouter une adresse e-mail",
        title: "Adresses e-mail"
      },
      enterpriseAccountsSection: {
        title: "Comptes entreprise"
      },
      headerTitle__account: "Compte",
      headerTitle__security: "S\xE9curit\xE9",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "R\xE9g\xE9n\xE9rer les codes",
          headerTitle: "Codes de r\xE9cup\xE9ration",
          subtitle__regenerate: "Obtenez de nouveaux codes de r\xE9cup\xE9ration s\xE9curis\xE9s. Les codes de r\xE9cup\xE9ration ant\xE9rieurs seront supprim\xE9s et ne pourront pas \xEAtre utilis\xE9s.",
          title__regenerate: "R\xE9g\xE9n\xE9rer les codes de r\xE9cup\xE9ration"
        },
        phoneCode: {
          actionLabel__setDefault: "D\xE9finir par d\xE9faut",
          destructiveActionLabel: "Supprimer le num\xE9ro de t\xE9l\xE9phone"
        },
        primaryButton: "Ajouter la v\xE9rification en deux \xE9tapes",
        title: "V\xE9rification en deux \xE9tapes",
        totp: {
          destructiveActionTitle: "D\xE9sactiver",
          headerTitle: "Application d'authentification"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "D\xE9finir le mot de passe",
        primaryButton__updatePassword: "Changer le mot de passe",
        title: "Mot de passe"
      },
      phoneNumbersSection: {
        destructiveAction: "Supprimer le num\xE9ro de t\xE9l\xE9phone",
        detailsAction__nonPrimary: "D\xE9finir comme principale",
        detailsAction__primary: "Compl\xE9ter la v\xE9rification",
        detailsAction__unverified: "Compl\xE9ter la v\xE9rification",
        primaryButton: "Ajouter un num\xE9ro de t\xE9l\xE9phone",
        title: "Les num\xE9ros de t\xE9l\xE9phone"
      },
      profileSection: {
        primaryButton: "",
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "D\xE9finir le nom d'utilisateur",
        primaryButton__updateUsername: "Changer le nom d'utilisateur",
        title: "Nom d'utilisateur"
      },
      web3WalletsSection: {
        destructiveAction: "Supprimer le portefeuille",
        primaryButton: "Portefeuilles Web3",
        title: "Portefeuilles Web3"
      }
    },
    usernamePage: {
      successMessage: "Votre nom d'utilisateur a \xE9t\xE9 mis \xE0 jour.",
      title__set: "Mettre \xE0 jour le nom d'utilisateur",
      title__update: "Mettre \xE0 jour le nom d'utilisateur"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus vous connecter avec ce portefeuille web3.",
        successMessage: "{{web3Wallet}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer le portefeuille Web3"
      },
      subtitle__availableWallets: "S\xE9lectionnez un portefeuille Web3 pour vous connecter \xE0 votre compte.",
      subtitle__unavailableWallets: "Il n'y a pas de portefeuilles Web3 disponibles.",
      successMessage: "Le portefeuille a \xE9t\xE9 ajout\xE9 \xE0 votre compte.",
      title: "Ajouter un portefeuille Web3"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  frFR
});
