// @clerk/localizations/dist/he-IL.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/he-IL.ts
var he_IL_exports = {};
__export(he_IL_exports, {
  heIL: () => heIL
});
module.exports = __toCommonJS(he_IL_exports);
var heIL = {
  locale: "he-IL",
  backButton: "\u05D7\u05D6\u05D5\u05E8",
  badge__default: "\u05D1\u05E8\u05D9\u05E8\u05EA \u05DE\u05D7\u05D3\u05DC",
  badge__otherImpersonatorDevice: "\u05DE\u05DB\u05E9\u05D9\u05E8 \u05DE\u05D7\u05E7\u05D4 \u05D0\u05D7\u05E8",
  badge__primary: "\u05E8\u05D0\u05E9\u05D9",
  badge__requiresAction: "\u05D3\u05D5\u05E8\u05E9 \u05E4\u05E2\u05D5\u05DC\u05D4",
  badge__thisDevice: "\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D6\u05D4",
  badge__unverified: "\u05DC\u05D0 \u05DE\u05D0\u05D5\u05DE\u05EA",
  badge__userDevice: "\u05DE\u05DB\u05E9\u05D9\u05E8 \u05DE\u05E9\u05EA\u05DE\u05E9",
  badge__you: "\u05D0\u05EA\u05D4",
  createOrganization: {
    formButtonSubmit: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    invitePage: {
      formButtonReset: "\u05D3\u05DC\u05D2"
    },
    title: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF"
  },
  dates: {
    lastDay: "\u05D0\u05EA\u05DE\u05D5\u05DC \u05D1-{{ date | timeString('iw') }}",
    next6Days: "{{ date | weekday('iw','long') }} \u05D1-{{ date | timeString('iw') }}",
    nextDay: "\u05DE\u05D7\u05E8 \u05D1-{{ date | timeString('iw') }}",
    numeric: "{{ date | numeric('iw') }}",
    previous6Days: "{{ date | weekday('iw','long') }} \u05D4\u05D0\u05D7\u05E8\u05D5\u05DF \u05D1-{{ date | timeString('iw') }}",
    sameDay: "\u05D4\u05D9\u05D5\u05DD \u05D1-{{ date | timeString('iw') }}"
  },
  dividerText: "\u05D0\u05D5",
  footerActionLink__useAnotherMethod: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA",
  footerPageLink__help: "\u05E2\u05D6\u05E8\u05D4",
  footerPageLink__privacy: "\u05E4\u05E8\u05D8\u05D9\u05D5\u05EA",
  footerPageLink__terms: "\u05EA\u05E0\u05D0\u05D9\u05DD",
  formButtonPrimary: "\u05D4\u05DE\u05E9\u05DA",
  formButtonPrimary__verify: "Verify",
  formFieldAction__forgotPassword: "\u05E9\u05DB\u05D7\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4?",
  formFieldError__matchingPasswords: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05EA\u05D5\u05D0\u05DE\u05D5\u05EA.",
  formFieldError__notMatchingPasswords: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05D0\u05D9\u05E0\u05DF \u05EA\u05D5\u05D0\u05DE\u05D5\u05EA.",
  formFieldError__verificationLinkExpired: "The verification link expired. Please request a new link.",
  formFieldHintText__optional: "\u05D0\u05D5\u05E4\u05E6\u05D9\u05D5\u05E0\u05DC\u05D9",
  formFieldHintText__slug: "A slug is a human-readable ID that must be unique. It\u2019s often used in URLs.",
  formFieldInputPlaceholder__backupCode: "",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Delete account",
  formFieldInputPlaceholder__emailAddress: "",
  formFieldInputPlaceholder__emailAddress_username: "",
  formFieldInputPlaceholder__emailAddresses: '\u05D4\u05DB\u05E0\u05E1 \u05D0\u05D5 \u05D4\u05D3\u05D1\u05E7 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D5 \u05DB\u05EA\u05D5\u05D1\u05D5\u05EA \u05D3\u05D5\u05D0"\u05DC, \u05DE\u05D5\u05E4\u05E8\u05D3\u05D5\u05EA \u05D1\u05E8\u05D5\u05D5\u05D7\u05D9\u05DD \u05D0\u05D5 \u05E4\u05E1\u05D9\u05E7\u05D9\u05DD',
  formFieldInputPlaceholder__firstName: "",
  formFieldInputPlaceholder__lastName: "",
  formFieldInputPlaceholder__organizationDomain: "",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "",
  formFieldInputPlaceholder__organizationName: "",
  formFieldInputPlaceholder__organizationSlug: "",
  formFieldInputPlaceholder__password: "",
  formFieldInputPlaceholder__phoneNumber: "",
  formFieldInputPlaceholder__username: "",
  formFieldLabel__automaticInvitations: "Enable automatic invitations for this domain",
  formFieldLabel__backupCode: "\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
  formFieldLabel__confirmDeletion: "Confirmation",
  formFieldLabel__confirmPassword: "\u05D0\u05DE\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4",
  formFieldLabel__currentPassword: "\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E0\u05D5\u05DB\u05D7\u05D9\u05EA",
  formFieldLabel__emailAddress: '\u05DB\u05EA\u05D5\u05D1\u05EA \u05D3\u05D5\u05D0"\u05DC',
  formFieldLabel__emailAddress_username: '\u05DB\u05EA\u05D5\u05D1\u05EA \u05D3\u05D5\u05D0"\u05DC \u05D0\u05D5 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9',
  formFieldLabel__emailAddresses: '\u05DB\u05EA\u05D5\u05D1\u05D5\u05EA \u05D3\u05D5\u05D0"\u05DC',
  formFieldLabel__firstName: "\u05E9\u05DD \u05E4\u05E8\u05D8\u05D9",
  formFieldLabel__lastName: "\u05E9\u05DD \u05DE\u05E9\u05E4\u05D7\u05D4",
  formFieldLabel__newPassword: "\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D7\u05D3\u05E9\u05D4",
  formFieldLabel__organizationDomain: "Domain",
  formFieldLabel__organizationDomainDeletePending: "Delete pending invitations and suggestions",
  formFieldLabel__organizationDomainEmailAddress: "Verification email address",
  formFieldLabel__organizationDomainEmailAddressDescription: "Enter an email address under this domain to receive a code and verify this domain.",
  formFieldLabel__organizationName: "\u05E9\u05DD \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF",
  formFieldLabel__organizationSlug: "\u05DB\u05EA\u05D5\u05D1\u05EA URL \u05E9\u05DC \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "\u05E1\u05D9\u05E1\u05DE\u05D4",
  formFieldLabel__phoneNumber: "\u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
  formFieldLabel__role: "\u05EA\u05E4\u05E7\u05D9\u05D3",
  formFieldLabel__signOutOfOtherSessions: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05DB\u05DC \u05D4\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05D4\u05D0\u05D7\u05E8\u05D9\u05DD",
  formFieldLabel__username: "\u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
  impersonationFab: {
    action__signOut: "\u05D4\u05EA\u05E0\u05EA\u05E7",
    title: "\u05DE\u05D7\u05D5\u05D1\u05E8 \u05DB{{identifier}}"
  },
  maintenanceMode: void 0,
  membershipRole__admin: "\u05DE\u05E0\u05D4\u05DC",
  membershipRole__basicMember: "\u05D7\u05D1\u05E8",
  membershipRole__guestMember: "\u05D0\u05D5\u05E8\u05D7",
  organizationList: {
    action__createOrganization: "Create organization",
    action__invitationAccept: "Join",
    action__suggestionsAccept: "Request to join",
    createOrganization: "Create Organization",
    invitationAcceptedLabel: "Joined",
    subtitle: "to continue to {{applicationName}}",
    suggestionsAcceptedLabel: "Pending approval",
    title: "Choose an account",
    titleWithoutPersonal: "Choose an organization"
  },
  organizationProfile: {
    badge__automaticInvitation: "Automatic invitations",
    badge__automaticSuggestion: "Automatic suggestions",
    badge__manualInvitation: "No automatic enrollment",
    badge__unverified: "Unverified",
    createDomainPage: {
      subtitle: "Add the domain to verify. Users with email addresses at this domain can join the organization automatically or request to join.",
      title: "Add domain"
    },
    invitePage: {
      detailsTitle__inviteFailed: "\u05DC\u05D0 \u05D4\u05D9\u05D4 \u05E0\u05D9\u05EA\u05DF \u05DC\u05E9\u05DC\u05D5\u05D7 \u05D0\u05EA \u05D4\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA. \u05EA\u05E7\u05DF \u05D0\u05EA \u05D4\u05D1\u05E2\u05D9\u05D5\u05EA \u05D4\u05D1\u05D0\u05D5\u05EA \u05D5\u05E0\u05E1\u05D4 \u05E9\u05D5\u05D1:",
      formButtonPrimary__continue: "\u05E9\u05DC\u05D7 \u05D4\u05D6\u05DE\u05E0\u05D5\u05EA",
      selectDropdown__role: "Select role",
      subtitle: "\u05D4\u05D6\u05DE\u05DF \u05D7\u05D1\u05E8\u05D9\u05DD \u05D7\u05D3\u05E9\u05D9\u05DD \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D6\u05D4",
      successMessage: "\u05D4\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05E0\u05E9\u05DC\u05D7\u05D5 \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4",
      title: "\u05D4\u05D6\u05DE\u05DF \u05D7\u05D1\u05E8\u05D9\u05DD"
    },
    membersPage: {
      action__invite: "\u05D4\u05D6\u05DE\u05DF",
      activeMembersTab: {
        menuAction__remove: "\u05D4\u05E1\u05E8 \u05D7\u05D1\u05E8",
        tableHeader__actions: "",
        tableHeader__joined: "\u05D4\u05E6\u05D8\u05E8\u05E3",
        tableHeader__role: "\u05EA\u05E4\u05E7\u05D9\u05D3",
        tableHeader__user: "\u05DE\u05E9\u05EA\u05DE\u05E9"
      },
      detailsTitle__emptyRow: "\u05D0\u05D9\u05DF \u05D7\u05D1\u05E8\u05D9\u05DD \u05DC\u05D4\u05E6\u05D2\u05D4",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invite users by connecting an email domain with your organization. Anyone who signs up with a matching email domain will be able to join the organization anytime.",
          headerTitle: "Automatic invitations",
          primaryButton: "Manage verified domains"
        },
        table__emptyRow: "No invitations to display"
      },
      invitedMembersTab: {
        menuAction__revoke: "\u05D1\u05D8\u05DC \u05D4\u05D6\u05DE\u05E0\u05D4",
        tableHeader__invited: "\u05D4\u05D5\u05D6\u05DE\u05DF"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Users who sign up with a matching email domain, will be able to see a suggestion to request to join your organization.",
          headerTitle: "Automatic suggestions",
          primaryButton: "Manage verified domains"
        },
        menuAction__approve: "Approve",
        menuAction__reject: "Reject",
        tableHeader__requested: "Requested access",
        table__emptyRow: "No requests to display"
      },
      start: {
        headerTitle__invitations: "Invitations",
        headerTitle__members: "Members",
        headerTitle__requests: "Requests"
      }
    },
    navbar: {
      description: "Manage your organization.",
      general: "General",
      members: "Members",
      title: "Organization"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Are you sure you want to delete this organization?",
          messageLine2: "This action is permanent and irreversible.",
          successMessage: "You have deleted the organization.",
          title: "Delete organization"
        },
        leaveOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "\u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05E2\u05D6\u05D5\u05D1 \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF \u05D4\u05D6\u05D4? \u05EA\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D6\u05D4 \u05D5\u05DC\u05D9\u05D9\u05E9\u05D5\u05DE\u05D9\u05DD \u05E9\u05DC\u05D5.",
          messageLine2: "\u05E4\u05E2\u05D5\u05DC\u05D4 \u05D6\u05D5 \u05D4\u05D9\u05D0 \u05E1\u05D5\u05E4\u05D9\u05EA \u05D5\u05D1\u05DC\u05EA\u05D9 \u05D4\u05E4\u05D9\u05DB\u05D4.",
          successMessage: "\u05E2\u05D6\u05D1\u05EA \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF.",
          title: "\u05E2\u05D6\u05D5\u05D1 \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF"
        },
        title: "\u05E1\u05D9\u05DB\u05D5\u05DF"
      },
      domainSection: {
        menuAction__manage: "Manage",
        menuAction__remove: "Delete",
        menuAction__verify: "Verify",
        primaryButton: "Add domain",
        subtitle: "Allow users to join the organization automatically or request to join based on a verified email domain.",
        title: "Verified domains"
      },
      successMessage: "\u05D4\u05D0\u05E8\u05D2\u05D5\u05DF \u05E2\u05D5\u05D3\u05DB\u05DF.",
      title: "\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC \u05D0\u05E8\u05D2\u05D5\u05DF"
    },
    removeDomainPage: {
      messageLine1: "The email domain {{domain}} will be removed.",
      messageLine2: "Users won\u2019t be able to join the organization automatically after this.",
      successMessage: "{{domain}} has been removed.",
      title: "Remove domain"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "\u05D7\u05D1\u05E8\u05D9\u05DD",
      profileSection: {
        primaryButton: "",
        title: "Organization Profile",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Removing this domain will affect invited users.",
        removeDomainActionLabel__remove: "Remove domain",
        removeDomainSubtitle: "Remove this domain from your verified domains",
        removeDomainTitle: "Remove domain"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Users are automatically invited to join the organization when they sign-up and can join anytime.",
        automaticInvitationOption__label: "Automatic invitations",
        automaticSuggestionOption__description: "Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.",
        automaticSuggestionOption__label: "Automatic suggestions",
        calloutInfoLabel: "Changing the enrollment mode will only affect new users.",
        calloutInvitationCountLabel: "Pending invitations sent to users: {{count}}",
        calloutSuggestionCountLabel: "Pending suggestions sent to users: {{count}}",
        manualInvitationOption__description: "Users can only be invited manually to the organization.",
        manualInvitationOption__label: "No automatic enrollment",
        subtitle: "Choose how users from this domain can join the organization."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Enrollment options"
      },
      subtitle: "The domain {{domain}} is now verified. Continue by selecting enrollment mode.",
      title: "Update {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Enter the verification code sent to your email address",
      formTitle: "Verification code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "The domain {{domainName}} needs to be verified via email.",
      subtitleVerificationCodeScreen: "A verification code was sent to {{emailAddress}}. Enter the code to continue.",
      title: "Verify domain"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    action__invitationAccept: "Join",
    action__manageOrganization: "\u05E0\u05D4\u05DC \u05D0\u05E8\u05D2\u05D5\u05DF",
    action__suggestionsAccept: "Request to join",
    notSelected: "\u05DC\u05D0 \u05E0\u05D1\u05D7\u05E8\u05D4 \u05D0\u05E8\u05D2\u05D5\u05DF",
    personalWorkspace: "\u05D0\u05D9\u05D6\u05D5\u05E8 \u05D0\u05D9\u05E9\u05D9",
    suggestionsAcceptedLabel: "Pending approval"
  },
  paginationButton__next: "\u05D4\u05D1\u05D0",
  paginationButton__previous: "\u05D4\u05E7\u05D5\u05D3\u05DD",
  paginationRowText__displaying: "\u05DE\u05E6\u05D9\u05D2",
  paginationRowText__of: "\u05DE\u05EA\u05D5\u05DA",
  signIn: {
    accountSwitcher: {
      action__addAccount: "Add account",
      action__signOutAll: "Sign out of all accounts",
      subtitle: "Select the account with which you wish to continue.",
      title: "Choose an account"
    },
    alternativeMethods: {
      actionLink: "\u05E7\u05D1\u05DC \u05E2\u05D6\u05E8\u05D4",
      actionText: "Don\u2019t have any of these?",
      blockButton__backupCode: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
      blockButton__emailCode: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC-{{identifier}}",
      blockButton__emailLink: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC-{{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "\u05D4\u05EA\u05D7\u05D1\u05E8 \u05E2\u05DD \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
      blockButton__phoneCode: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 SMS \u05DC-{{identifier}}",
      blockButton__totp: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA",
      getHelp: {
        blockButton__emailSupport: "\u05DE\u05D9\u05D9\u05DC \u05DC\u05EA\u05DE\u05D9\u05DB\u05D4",
        content: "\u05D0\u05DD \u05D0\u05EA\u05D4 \u05E0\u05EA\u05E7\u05DC \u05D1\u05E7\u05E9\u05D9\u05D9\u05DD \u05D1\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA \u05DC\u05D7\u05E9\u05D1\u05D5\u05E0\u05DA, \u05E9\u05DC\u05D7 \u05DC\u05E0\u05D5 \u05DE\u05D9\u05D9\u05DC \u05D5\u05E0\u05E2\u05D1\u05D5\u05D3 \u05D0\u05D9\u05EA\u05DA \u05DB\u05D3\u05D9 \u05DC\u05E9\u05D7\u05D6\u05E8 \u05D0\u05EA \u05D4\u05D2\u05D9\u05E9\u05D4 \u05D1\u05D4\u05E7\u05D3\u05DD \u05D4\u05D0\u05E4\u05E9\u05E8\u05D9.",
        title: "\u05E7\u05D1\u05DC \u05E2\u05D6\u05E8\u05D4"
      },
      subtitle: "Facing issues? You can use any of these methods to sign in.",
      title: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA"
    },
    backupCodeMfa: {
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D4\u05DB\u05E0\u05E1 \u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9"
    },
    emailCode: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA'
    },
    emailLink: {
      expired: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E8\u05D9\u05EA \u05DC\u05D4\u05DE\u05E9\u05DA.",
        title: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D6\u05D4 \u05E4\u05D2 \u05EA\u05D5\u05E7\u05E3"
      },
      failed: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E8\u05D9\u05EA \u05DC\u05D4\u05DE\u05E9\u05DA.",
        title: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D6\u05D4 \u05DC\u05D0 \u05D7\u05D5\u05E7\u05D9"
      },
      formSubtitle: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      formTitle: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA",
      loading: {
        subtitle: "\u05EA\u05D5\u05E2\u05D1\u05E8 \u05D1\u05E7\u05E8\u05D5\u05D1",
        title: "\u05DE\u05EA\u05D7\u05D1\u05E8..."
      },
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      unusedTab: {
        title: "\u05D0\u05EA\u05D4 \u05D9\u05DB\u05D5\u05DC \u05DC\u05E1\u05D2\u05D5\u05E8 \u05D0\u05EA \u05D4\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05D6\u05D5"
      },
      verified: {
        subtitle: "\u05EA\u05D5\u05E2\u05D1\u05E8 \u05D1\u05E7\u05E8\u05D5\u05D1",
        title: "\u05E0\u05DB\u05E0\u05E1\u05EA \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4"
      },
      verifiedSwitchTab: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E8\u05D9\u05EA \u05DC\u05D4\u05DE\u05E9\u05DA",
        subtitleNewTab: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05E9\u05E0\u05E4\u05EA\u05D7\u05D4 \u05D7\u05D3\u05E9\u05D4 \u05DC\u05D4\u05DE\u05E9\u05DA",
        titleNewTab: "\u05E0\u05DB\u05E0\u05E1 \u05D1\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D0\u05D7\u05E8\u05EA"
      }
    },
    forgotPassword: {
      formTitle: "\u05D0\u05E4\u05E1 \u05E7\u05D5\u05D3 \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "to reset your password",
      subtitle_email: "First, enter the code sent to your email ID",
      subtitle_phone: "First, enter the code sent to your phone",
      title: "Reset password"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "\u05D0\u05E4\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
      label__alternativeMethods: "\u05D0\u05D5, \u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA.",
      title: "\u05E9\u05DB\u05D7\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4?"
    },
    noAvailableMethods: {
      message: "Cannot proceed with sign in. There's no available authentication factor.",
      subtitle: "An error occurred",
      title: "Cannot sign in"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E1\u05D9\u05E1\u05DE\u05EA\u05DA"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    phoneCodeMfa: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "",
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    resetPassword: {
      formButtonPrimary: "\u05D0\u05E4\u05E1 \u05E1\u05D9\u05E1\u05DE\u05D4",
      requiredMessage: "For security reasons, it is required to reset your password.",
      successMessage: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05E9\u05D5\u05E0\u05EA\u05D4 \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4. \u05DE\u05D7\u05D1\u05E8 \u05D0\u05D5\u05EA\u05DA, \u05D0\u05E0\u05D0 \u05D4\u05DE\u05EA\u05DF \u05E8\u05D2\u05E2.",
      title: "\u05D0\u05E4\u05E1 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    resetPasswordMfa: {
      detailsLabel: "\u05D0\u05E0\u05D7\u05E0\u05D5 \u05E6\u05E8\u05D9\u05DB\u05D9\u05DD \u05DC\u05D0\u05DE\u05EA \u05D0\u05EA \u05D6\u05D4\u05D5\u05EA\u05DA \u05DC\u05E4\u05E0\u05D9 \u05E9\u05E0\u05D0\u05E4\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA."
    },
    start: {
      actionLink: "\u05D4\u05E8\u05E9\u05DE\u05D4",
      actionLink__use_email: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D3\u05D5\u05D0"\u05DC',
      actionLink__use_email_username: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D3\u05D5\u05D0"\u05DC \u05D0\u05D5 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9',
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D8\u05DC\u05E4\u05D5\u05DF",
      actionLink__use_username: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
      actionText: "\u05D0\u05D9\u05DF \u05DC\u05DA \u05D7\u05E9\u05D1\u05D5\u05DF?",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D4\u05EA\u05D7\u05D1\u05E8"
    },
    totpMfa: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      subtitle: "",
      title: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05D9 \u05E9\u05DC\u05D1\u05D9\u05DD"
    }
  },
  signInEnterPasswordTitle: "\u05D4\u05D6\u05DF \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
  signUp: {
    continue: {
      actionLink: "\u05D4\u05EA\u05D7\u05D1\u05E8",
      actionText: "\u05D9\u05E9 \u05DC\u05DA \u05D7\u05E9\u05D1\u05D5\u05DF?",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05DE\u05DC\u05D0 \u05E9\u05D3\u05D5\u05EA \u05D7\u05E1\u05E8\u05D9\u05DD"
    },
    emailCode: {
      formSubtitle: '\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D0\u05DE\u05EA \u05D0\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA'
    },
    emailLink: {
      formSubtitle: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      formTitle: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA",
      loading: {
        title: "\u05DE\u05EA\u05D7\u05D9\u05DC \u05DC\u05D4\u05D9\u05E8\u05E9\u05DD..."
      },
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D0\u05DE\u05EA \u05D0\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      verified: {
        title: "\u05E0\u05E8\u05E9\u05DE\u05EA \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4"
      },
      verifiedSwitchTab: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05E9\u05E0\u05E4\u05EA\u05D7\u05D4 \u05DC\u05D0\u05D7\u05E8\u05D5\u05E0\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA",
        subtitleNewTab: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05E7\u05D5\u05D3\u05DE\u05EA \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA",
        title: '\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5\u05D0"\u05DC \u05D4\u05E6\u05DC\u05D9\u05D7'
      }
    },
    phoneCode: {
      formSubtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA",
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D0\u05DE\u05EA \u05D0\u05EA \u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    start: {
      actionLink: "\u05D4\u05EA\u05D7\u05D1\u05E8",
      actionText: "\u05D9\u05E9 \u05DC\u05DA \u05D7\u05E9\u05D1\u05D5\u05DF?",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05E6\u05D5\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA"
    }
  },
  socialButtonsBlockButton: "\u05D4\u05DE\u05E9\u05DA \u05E2\u05DD {{provider|titleize}}",
  unstable__errors: {
    captcha_invalid: "Sign up unsuccessful due to failed security validations. Please refresh the page to try again or reach out to support for more assistance.",
    captcha_unavailable: "Sign up unsuccessful due to failed bot validation. Please refresh the page to try again or reach out to support for more assistance.",
    form_code_incorrect: "",
    form_identifier_exists: "",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "",
    form_param_format_invalid: "",
    form_param_format_invalid__email_address: "Email address must be a valid email address.",
    form_param_format_invalid__phone_number: "Phone number must be in a valid international format",
    form_param_max_length_exceeded__first_name: "First name should not exceed 256 characters.",
    form_param_max_length_exceeded__last_name: "Last name should not exceed 256 characters.",
    form_param_max_length_exceeded__name: "Name should not exceed 256 characters.",
    form_param_nil: "",
    form_password_incorrect: "",
    form_password_length_too_short: "",
    form_password_not_strong_enough: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D0\u05D9\u05E0\u05D4 \u05DE\u05E1\u05E4\u05D9\u05E7 \u05D7\u05D6\u05E7\u05D4.",
    form_password_pwned: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D4\u05D6\u05D5 \u05E0\u05DE\u05E6\u05D0\u05D4 \u05DB\u05D7\u05DC\u05E7 \u05DE\u05D4\u05E4\u05E8\u05D8\u05D9\u05DD \u05E9\u05E0\u05D7\u05E9\u05E4\u05D5 \u05D1\u05D4\u05E4\u05E8\u05EA \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D5\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D4, \u05E0\u05E1\u05D4 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05D0\u05D7\u05E8\u05EA \u05D1\u05DE\u05E7\u05D5\u05DD.",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D7\u05D5\u05E8\u05D2\u05EA \u05DE\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D1\u05D9\u05D9\u05D8\u05D9\u05DD \u05D4\u05DE\u05E8\u05D1\u05D9 \u05D4\u05DE\u05D5\u05EA\u05E8, \u05E0\u05E1\u05D4 \u05DC\u05E7\u05E6\u05E8 \u05D0\u05D5\u05EA\u05D4 \u05D0\u05D5 \u05DC\u05D4\u05E1\u05D9\u05E8 \u05DB\u05DE\u05D4 \u05EA\u05D5\u05D5\u05D9\u05DD \u05DE\u05D9\u05D5\u05D7\u05D3\u05D9\u05DD.",
    form_password_validation_failed: "\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05D2\u05D5\u05D9\u05D4",
    form_username_invalid_character: "",
    form_username_invalid_length: "",
    identification_deletion_failed: "You cannot delete your last identification.",
    not_allowed_access: "",
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "\u05E4\u05D7\u05D5\u05EA \u05DE-{{length}} \u05EA\u05D5\u05D5\u05D9\u05DD",
      minimumLength: "{{length}} \u05EA\u05D5\u05D5\u05D9\u05DD \u05D0\u05D5 \u05D9\u05D5\u05EA\u05E8",
      requireLowercase: "\u05D0\u05D5\u05EA \u05E7\u05D8\u05E0\u05D4",
      requireNumbers: "\u05DE\u05E1\u05E4\u05E8",
      requireSpecialCharacter: "\u05EA\u05D5 \u05DE\u05D9\u05D5\u05D7\u05D3",
      requireUppercase: "\u05D0\u05D5\u05EA \u05D2\u05D3\u05D5\u05DC\u05D4",
      sentencePrefix: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D7\u05D9\u05D9\u05D1\u05EA \u05DC\u05D4\u05DB\u05D9\u05DC"
    },
    phone_number_exists: "This phone number is taken. Please try another.",
    zxcvbn: {
      couldBeStronger: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05EA\u05E7\u05E4\u05D4, \u05D0\u05DA \u05D9\u05DB\u05D5\u05DC\u05D4 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D7\u05D6\u05E7\u05D4 \u05D9\u05D5\u05EA\u05E8. \u05E0\u05E1\u05D4 \u05DC\u05D4\u05D5\u05E1\u05D9\u05E3 \u05D9\u05D5\u05EA\u05E8 \u05EA\u05D5\u05D5\u05D9\u05DD.",
      goodPassword: "\u05E2\u05D1\u05D5\u05D3\u05D4 \u05D8\u05D5\u05D1\u05D4. \u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05DE\u05E6\u05D5\u05D9\u05E0\u05EA.",
      notEnough: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D0\u05D9\u05E0\u05D4 \u05DE\u05E1\u05E4\u05D9\u05E7 \u05D7\u05D6\u05E7\u05D4.",
      suggestions: {
        allUppercase: "\u05D4\u05D2\u05D3\u05DC \u05DB\u05DE\u05D4, \u05D0\u05DA \u05DC\u05D0 \u05D0\u05EA \u05DB\u05DC \u05D4\u05D0\u05D5\u05EA\u05D9\u05D5\u05EA.",
        anotherWord: "\u05D4\u05D5\u05E1\u05E3 \u05E2\u05D5\u05D3 \u05DE\u05D9\u05DC\u05D9\u05DD \u05E9\u05D4\u05DF \u05E4\u05D7\u05D5\u05EA \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA.",
        associatedYears: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05E9\u05E0\u05D9\u05DD \u05E9\u05E7\u05E9\u05D5\u05E8\u05D5\u05EA \u05D0\u05DC\u05D9\u05DA.",
        capitalization: "\u05D4\u05D2\u05D3\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DE\u05D0\u05D5\u05EA \u05D0\u05D7\u05EA.",
        dates: "\u05D4\u05DE\u05E0\u05E2 \u05DE\u05EA\u05D0\u05E8\u05D9\u05DB\u05D9\u05DD \u05D5\u05E9\u05E0\u05D9\u05DD \u05E9\u05E7\u05E9\u05D5\u05E8\u05D9\u05DD \u05D0\u05DC\u05D9\u05DA.",
        l33t: "\u05D4\u05DE\u05E0\u05E2 \u05DE\u05D4\u05D7\u05DC\u05E4\u05D5\u05EA \u05EA\u05D5\u05D5\u05D9\u05DD \u05E6\u05E4\u05D5\u05D9\u05D5\u05EA \u05DB\u05DE\u05D5 '@' \u05D1\u05DE\u05E7\u05D5\u05DD 'a'.",
        longerKeyboardPattern: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D3\u05E4\u05D5\u05E1\u05D9 \u05DE\u05E7\u05DC\u05D3\u05EA \u05D0\u05E8\u05D5\u05DB\u05D9\u05DD \u05D9\u05D5\u05EA\u05E8 \u05D5\u05E9\u05E0\u05D4 \u05D0\u05EA \u05DB\u05D9\u05D5\u05D5\u05DF \u05D4\u05D4\u05E7\u05DC\u05D3\u05D4 \u05DE\u05E1\u05E4\u05E8 \u05E4\u05E2\u05DE\u05D9\u05DD.",
        noNeed: "\u05D0\u05EA\u05D4 \u05D9\u05DB\u05D5\u05DC \u05DC\u05D9\u05E6\u05D5\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05D7\u05D6\u05E7\u05D5\u05EA \u05DC\u05DC\u05D0 \u05E9\u05D9\u05DE\u05D5\u05E9 \u05D1\u05E1\u05DE\u05DC\u05D9\u05DD, \u05DE\u05E1\u05E4\u05E8\u05D9\u05DD, \u05D0\u05D5 \u05D0\u05D5\u05EA\u05D9\u05D5\u05EA \u05D2\u05D3\u05D5\u05DC\u05D5\u05EA.",
        pwned: "\u05D0\u05DD \u05D0\u05EA\u05D4 \u05DE\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D6\u05D5 \u05D1\u05DE\u05E7\u05D5\u05DE\u05D5\u05EA \u05D0\u05D7\u05E8\u05D9\u05DD, \u05E2\u05DC\u05D9\u05DA \u05DC\u05E9\u05E0\u05D5\u05EA\u05D4.",
        recentYears: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05E9\u05E0\u05D9\u05DD \u05D0\u05D7\u05E8\u05D5\u05E0\u05D5\u05EA.",
        repeated: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05DE\u05D9\u05DC\u05D9\u05DD \u05D5\u05EA\u05D5\u05D5\u05D9\u05DD \u05DE\u05D5\u05D7\u05D6\u05E8\u05D9\u05DD.",
        reverseWords: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05DB\u05EA\u05D9\u05D1\u05D4 \u05D4\u05E4\u05D5\u05DB\u05D4 \u05E9\u05DC \u05DE\u05D9\u05DC\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA.",
        sequences: "\u05D4\u05DE\u05E0\u05E2 \u05DE\u05E8\u05E6\u05E4\u05D5\u05EA \u05EA\u05D5\u05D5\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA.",
        useWords: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05DE\u05D9\u05DC\u05D9\u05DD \u05DE\u05E8\u05D5\u05D1\u05D5\u05EA, \u05D0\u05DA \u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05D1\u05D9\u05D8\u05D5\u05D9\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D9\u05DD."
      },
      warnings: {
        common: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05E0\u05E4\u05D5\u05E5.",
        commonNames: "\u05E9\u05DE\u05D5\u05EA \u05E0\u05E4\u05D5\u05E6\u05D9\u05DD \u05D5\u05E9\u05DE\u05D5\u05EA \u05DE\u05E9\u05E4\u05D7\u05D4 \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        dates: "\u05EA\u05D0\u05E8\u05D9\u05DB\u05D9\u05DD \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        extendedRepeat: '\u05D3\u05E4\u05D5\u05E1\u05D9\u05DD \u05DE\u05D5\u05D7\u05D6\u05E8\u05D9\u05DD \u05E9\u05DC \u05EA\u05D5\u05D5\u05D9\u05DD \u05DB\u05DE\u05D5 "abcabcabc" \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.',
        keyPattern: "\u05D3\u05E4\u05D5\u05E1\u05D9\u05DD \u05E7\u05E6\u05E8\u05D9\u05DD \u05E9\u05DC \u05DE\u05E7\u05DC\u05D3\u05EA \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        namesByThemselves: "\u05E9\u05DE\u05D5\u05EA \u05D1\u05D5\u05D3\u05D3\u05D9\u05DD \u05D0\u05D5 \u05E9\u05DE\u05D5\u05EA \u05DE\u05E9\u05E4\u05D7\u05D4 \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        pwned: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D4\u05D5\u05D7\u05E9\u05E4\u05D4 \u05D1\u05DE\u05D4\u05DC\u05DA \u05D4\u05E4\u05E8\u05EA \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D1\u05D0\u05D9\u05E0\u05D8\u05E8\u05E0\u05D8.",
        recentYears: "\u05E9\u05E0\u05D9\u05DD \u05D0\u05D7\u05E8\u05D5\u05E0\u05D5\u05EA \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        sequences: '\u05E8\u05E6\u05E4\u05D5\u05EA \u05EA\u05D5\u05D5\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA \u05DB\u05DE\u05D5 "abc" \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.',
        similarToCommon: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05D3\u05D5\u05DE\u05D4 \u05DC\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05E0\u05E4\u05D5\u05E5.",
        simpleRepeat: '\u05EA\u05D5\u05D5\u05D9\u05DD \u05DE\u05D5\u05D7\u05D6\u05E8\u05D9\u05DD \u05DB\u05DE\u05D5 "aaa" \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.',
        straightRow: "\u05E9\u05D5\u05E8\u05D5\u05EA \u05D9\u05E9\u05E8\u05D5\u05EA \u05E9\u05DC \u05DE\u05E7\u05E9\u05D9\u05DD \u05D1\u05DE\u05E7\u05DC\u05D3\u05EA \u05E9\u05DC\u05DA \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        topHundred: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05EA\u05DB\u05D5\u05E3.",
        topTen: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05DB\u05D1\u05D3.",
        userInputs: "\u05D0\u05D9\u05DF \u05DC\u05D4\u05D6\u05D9\u05DF \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D0\u05D9\u05E9\u05D9\u05D9\u05DD \u05D0\u05D5 \u05E7\u05E9\u05D5\u05E8\u05D9\u05DD \u05DC\u05D3\u05E3.",
        wordByItself: "\u05DE\u05D9\u05DC\u05D9\u05DD \u05D1\u05D5\u05D3\u05D3\u05D5\u05EA \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9."
      }
    }
  },
  userButton: {
    action__addAccount: "\u05D4\u05D5\u05E1\u05E3 \u05D7\u05E9\u05D1\u05D5\u05DF",
    action__manageAccount: "\u05E0\u05D4\u05DC \u05D7\u05E9\u05D1\u05D5\u05DF",
    action__signOut: "\u05D4\u05EA\u05E0\u05EA\u05E7",
    action__signOutAll: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05DB\u05DC \u05D4\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA"
  },
  userProfile: {
    backupCodePage: {
      actionLabel__copied: "\u05D4\u05D5\u05E2\u05EA\u05E7!",
      actionLabel__copy: "\u05D4\u05E2\u05EA\u05E7 \u05D4\u05DB\u05DC",
      actionLabel__download: "\u05D4\u05D5\u05E8\u05D3 .txt",
      actionLabel__print: "\u05D4\u05D3\u05E4\u05E1",
      infoText1: "\u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05D9\u05D4\u05D9\u05D5 \u05DE\u05D5\u05E4\u05E2\u05DC\u05D9\u05DD \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05D6\u05D4.",
      infoText2: "\u05E9\u05DE\u05D5\u05E8 \u05D0\u05EA \u05E7\u05D5\u05D3\u05D9 \u05D4\u05D2\u05D9\u05D1\u05D5\u05D9 \u05D1\u05E1\u05D5\u05D3 \u05D5\u05D0\u05D7\u05E1\u05DF \u05D0\u05D5\u05EA\u05DD \u05D1\u05D1\u05D8\u05D7\u05D4. \u05EA\u05D5\u05DB\u05DC \u05DC\u05D7\u05D5\u05DC\u05DC \u05DE\u05D7\u05D3\u05E9 \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05D0\u05DD \u05D0\u05EA\u05D4 \u05D7\u05D5\u05E9\u05D3 \u05E9\u05D4\u05DD \u05E0\u05E4\u05D2\u05E2\u05D5.",
      subtitle__codelist: "\u05D0\u05D7\u05E1\u05DF \u05D0\u05D5\u05EA\u05DD \u05D1\u05D1\u05D8\u05D7\u05D4 \u05D5\u05E9\u05DE\u05D5\u05E8 \u05E2\u05DC\u05D9\u05D4\u05DD \u05D1\u05E1\u05D5\u05D3.",
      successMessage: "\u05E7\u05D5\u05D3\u05D9 \u05D4\u05D2\u05D9\u05D1\u05D5\u05D9 \u05DE\u05D5\u05E4\u05E2\u05DC\u05D9\u05DD \u05DB\u05E2\u05EA. \u05EA\u05D5\u05DB\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05D7\u05D3 \u05DE\u05D0\u05DC\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA, \u05D0\u05DD \u05D0\u05EA\u05D4 \u05DE\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA. \u05DB\u05DC \u05E7\u05D5\u05D3 \u05D9\u05DB\u05D5\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D5 \u05E8\u05E7 \u05E4\u05E2\u05DD \u05D0\u05D7\u05EA.",
      successSubtitle: "\u05EA\u05D5\u05DB\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05D7\u05D3 \u05DE\u05D0\u05DC\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA, \u05D0\u05DD \u05D0\u05EA\u05D4 \u05DE\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
      title__codelist: "\u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9"
    },
    connectedAccountPage: {
      formHint: "\u05D1\u05D7\u05E8 \u05E1\u05E4\u05E7 \u05DB\u05D3\u05D9 \u05DC\u05D7\u05D1\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      formHint__noAccounts: "\u05D0\u05D9\u05DF \u05E1\u05E4\u05E7\u05D9\u05DD \u05D6\u05DE\u05D9\u05E0\u05D9\u05DD \u05DC\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA \u05D7\u05D9\u05E6\u05D5\u05E0\u05D9\u05D9\u05DD.",
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D7\u05E9\u05D1\u05D5\u05DF \u05DE\u05D7\u05D5\u05D1\u05E8 \u05D6\u05D4 \u05D5\u05DB\u05DC \u05D4\u05EA\u05DB\u05D5\u05E0\u05D5\u05EA \u05D4\u05EA\u05DC\u05D5\u05D9\u05D5\u05EA \u05DC\u05D0 \u05D9\u05E2\u05D1\u05D3\u05D5.",
        successMessage: "{{connectedAccount}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF \u05DE\u05D7\u05D5\u05D1\u05E8"
      },
      socialButtonsBlockButton: "\u05D7\u05D1\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF {{provider|titleize}}",
      successMessage: "\u05D4\u05E1\u05E4\u05E7 \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D7\u05E9\u05D1\u05D5\u05DF \u05DE\u05D7\u05D5\u05D1\u05E8"
    },
    deletePage: {
      actionDescription: 'Type "Delete account" below to continue.',
      confirm: "Delete account",
      messageLine1: "Are you sure you want to delete your account?",
      messageLine2: "This action is permanent and irreversible.",
      title: "Delete account"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DE\u05DB\u05D9\u05DC \u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D9\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D6\u05D5.",
        formSubtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC-{{identifier}}",
        formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
        resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05DE\u05D7\u05D3\u05E9",
        successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{identifier}} \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA."
      },
      emailLink: {
        formHint: "\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DE\u05DB\u05D9\u05DC \u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05D9\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D6\u05D5.",
        formSubtitle: "\u05DC\u05D7\u05E5 \u05E2\u05DC \u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC-{{identifier}}",
        formTitle: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA",
        resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05DE\u05D7\u05D3\u05E9",
        successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{identifier}} \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA."
      },
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D6\u05D5.",
        successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{emailAddress}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC"
      },
      title: "\u05D4\u05D5\u05E1\u05E3 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
      verifyTitle: "Verify email address"
    },
    formButtonPrimary__add: "Add",
    formButtonPrimary__continue: "\u05D4\u05DE\u05E9\u05DA",
    formButtonPrimary__finish: "\u05E1\u05D9\u05D9\u05DD",
    formButtonPrimary__remove: "Remove",
    formButtonPrimary__save: "Save",
    formButtonReset: "\u05D1\u05D8\u05DC",
    mfaPage: {
      formHint: "\u05D1\u05D7\u05E8 \u05E9\u05D9\u05D8\u05D4 \u05DC\u05D4\u05D5\u05E1\u05E4\u05D4.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
    },
    mfaPhoneCodePage: {
      backButton: "Use existing number",
      primaryButton__addPhoneNumber: "\u05D4\u05D5\u05E1\u05E3 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
      removeResource: {
        messageLine1: "{{identifier}} \u05DC\u05D0 \u05D9\u05E7\u05D1\u05DC \u05D9\u05D5\u05EA\u05E8 \u05E7\u05D5\u05D3\u05D9\u05DD \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA.",
        messageLine2: "\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA \u05E2\u05E9\u05D5\u05D9 \u05DC\u05D0 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D1\u05D8\u05D5\u05D7 \u05DB\u05DE\u05D5 \u05E9\u05D4\u05D5\u05D0. \u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA?",
        successMessage: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9 \u05D4\u05D5\u05E1\u05E8 \u05DC{{mfaPhoneCode}}",
        title: "\u05D4\u05E1\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
      },
      subtitle__availablePhoneNumbers: "\u05D1\u05D7\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05DC\u05D4\u05E8\u05E9\u05DE\u05D4 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9.",
      subtitle__unavailablePhoneNumbers: "\u05D0\u05D9\u05DF \u05DE\u05E1\u05E4\u05E8\u05D9 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05DE\u05D9\u05E0\u05D9\u05DD \u05DC\u05D4\u05E8\u05E9\u05DE\u05D4 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9.",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS code verification enabled",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "\u05E1\u05E8\u05D5\u05E7 \u05E7\u05D5\u05D3 QR \u05D1\u05DE\u05E7\u05D5\u05DD",
        buttonUnableToScan__nonPrimary: "\u05DC\u05D0 \u05D9\u05DB\u05D5\u05DC \u05DC\u05E1\u05E8\u05D5\u05E7 \u05E7\u05D5\u05D3 QR?",
        infoText__ableToScan: "\u05D4\u05D2\u05D3\u05E8 \u05E9\u05D9\u05D8\u05EA \u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA \u05D7\u05D3\u05E9\u05D4 \u05D1\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA \u05D5\u05E1\u05E8\u05D5\u05E7 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4-QR \u05D4\u05D1\u05D0 \u05DB\u05D3\u05D9 \u05DC\u05D7\u05D1\u05E8\u05D5 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        infoText__unableToScan: "\u05D4\u05D2\u05D3\u05E8 \u05E9\u05D9\u05D8\u05EA \u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA \u05D7\u05D3\u05E9\u05D4 \u05D1\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA \u05D5\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05D4\u05DE\u05E4\u05EA\u05D7 \u05E9\u05E1\u05D5\u05E4\u05E7 \u05DC\u05DE\u05D8\u05D4.",
        inputLabel__unableToScan1: "\u05D5\u05D5\u05D3\u05D0 \u05E9\u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05DE\u05D1\u05D5\u05E1\u05E1\u05D5\u05EA-\u05D6\u05DE\u05DF \u05D0\u05D5 \u05D7\u05D3-\u05E4\u05E2\u05DE\u05D9\u05D5\u05EA \u05DE\u05D5\u05E4\u05E2\u05DC\u05D5\u05EA, \u05D5\u05D0\u05D6 \u05E1\u05D9\u05D9\u05DD \u05DC\u05D7\u05D1\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        inputLabel__unableToScan2: "\u05DC\u05D7\u05DC\u05D5\u05E4\u05D9\u05DF, \u05D0\u05DD \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA \u05EA\u05D5\u05DE\u05DA \u05D1\u05DB\u05EA\u05D5\u05D1\u05EA URI \u05E9\u05DC TOTP, \u05EA\u05D5\u05DB\u05DC \u05D2\u05DD \u05DC\u05D4\u05E2\u05EA\u05D9\u05E7 \u05D0\u05EA \u05D4\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05DE\u05DC\u05D0\u05D4."
      },
      removeResource: {
        messageLine1: "\u05E7\u05D5\u05D3\u05D9 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05D4\u05D6\u05D4 \u05DC\u05D0 \u05D9\u05D4\u05D9\u05D5 \u05E0\u05D3\u05E8\u05E9\u05D9\u05DD \u05D9\u05D5\u05EA\u05E8 \u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA.",
        messageLine2: "\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA \u05E2\u05E9\u05D5\u05D9 \u05DC\u05D0 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D1\u05D8\u05D5\u05D7 \u05DB\u05DE\u05D5 \u05E9\u05D4\u05D5\u05D0. \u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA?",
        successMessage: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05EA \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05D4\u05D5\u05E1\u05E8.",
        title: "\u05D4\u05E1\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
      },
      successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9 \u05DE\u05D5\u05E4\u05E2\u05DC \u05DB\u05E2\u05EA. \u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA, \u05EA\u05D9\u05D3\u05E8\u05E9 \u05DC\u05D4\u05DB\u05E0\u05D9\u05E1 \u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05D6\u05D4 \u05DB\u05E9\u05DC\u05D1 \u05E0\u05D5\u05E1\u05E3.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D0\u05D9\u05DE\u05D5\u05EA",
      verifySubtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05D5\u05E6\u05E8 \u05E2\u05DC \u05D9\u05D3\u05D9 \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA",
      verifyTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA"
    },
    mobileButton__menu: "\u05EA\u05E4\u05E8\u05D9\u05D8",
    navbar: {
      account: "Profile",
      description: "Manage your account info.",
      security: "Security",
      title: "Account"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "It is recommended to sign out of all other devices which may have used your old password.",
      readonly: "Your password can currently not be edited because you can sign in only via the enterprise connection.",
      successMessage__set: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D4\u05D5\u05D2\u05D3\u05E8\u05D4.",
      successMessage__signOutOfOtherSessions: "\u05DB\u05DC \u05D4\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05D4\u05D0\u05D7\u05E8\u05D9\u05DD \u05D4\u05EA\u05E0\u05EA\u05E7\u05D5.",
      successMessage__update: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05E2\u05D5\u05D3\u05DB\u05E0\u05D4.",
      title__set: "\u05D4\u05D2\u05D3\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D4",
      title__update: "\u05E9\u05E0\u05D4 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    phoneNumberPage: {
      infoText: "\u05D4\u05D5\u05D3\u05E2\u05EA \u05D8\u05E7\u05E1\u05D8 \u05E9\u05DE\u05DB\u05D9\u05DC\u05D4 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05EA\u05D9\u05E9\u05DC\u05D7 \u05DC\u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05D4.",
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05D4.",
        successMessage: "{{phoneNumber}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF"
      },
      successMessage: "{{identifier}} \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
      verifySubtitle: "Enter the verification code sent to {{identifier}}",
      verifyTitle: "Verify phone number"
    },
    profilePage: {
      fileDropAreaHint: '\u05D4\u05E2\u05DC\u05D4 \u05EA\u05DE\u05D5\u05E0\u05D4 \u05D1\u05E4\u05D5\u05E8\u05DE\u05D8 JPG, PNG, GIF, \u05D0\u05D5 WEBP \u05D4\u05E7\u05D8\u05E0\u05D4 \u05DE-10 \u05DE"\u05D1',
      imageFormDestructiveActionSubtitle: "\u05D4\u05E1\u05E8 \u05EA\u05DE\u05D5\u05E0\u05D4",
      imageFormSubtitle: "\u05D4\u05E2\u05DC\u05D4 \u05EA\u05DE\u05D5\u05E0\u05D4",
      imageFormTitle: "\u05EA\u05DE\u05D5\u05E0\u05EA \u05E4\u05E8\u05D5\u05E4\u05D9\u05DC",
      readonly: "Your profile information has been provided by the enterprise connection and cannot be edited.",
      successMessage: "\u05D4\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC \u05E9\u05DC\u05DA \u05E2\u05D5\u05D3\u05DB\u05DF.",
      title: "\u05E2\u05D3\u05DB\u05DF \u05E4\u05E8\u05D5\u05E4\u05D9\u05DC"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05D4\u05DE\u05DB\u05E9\u05D9\u05E8",
        title: "\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05E4\u05E2\u05D9\u05DC\u05D9\u05DD"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "\u05E0\u05E1\u05D4 \u05E9\u05D5\u05D1",
        actionLabel__reauthorize: "\u05D0\u05E9\u05E8 \u05E2\u05DB\u05E9\u05D9\u05D5",
        destructiveActionTitle: "\u05D4\u05E1\u05E8",
        primaryButton: "\u05D7\u05D1\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF",
        subtitle__reauthorize: "The required scopes have been updated, and you may be experiencing limited functionality. Please re-authorize this application to avoid any issues",
        title: "\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA \u05DE\u05D7\u05D5\u05D1\u05E8\u05D9\u05DD"
      },
      dangerSection: {
        deleteAccountButton: "\u05DE\u05D7\u05E7 \u05D7\u05E9\u05D1\u05D5\u05DF",
        title: "\u05DE\u05E1\u05D5\u05DB\u05DF"
      },
      emailAddressesSection: {
        destructiveAction: "\u05D4\u05E1\u05E8 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
        detailsAction__nonPrimary: "\u05D4\u05D2\u05D3\u05E8 \u05DB\u05E2\u05D9\u05E7\u05E8\u05D9\u05EA",
        detailsAction__primary: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        detailsAction__unverified: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
        title: "\u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC"
      },
      enterpriseAccountsSection: {
        title: "\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA \u05D0\u05E8\u05D2\u05D5\u05E0\u05D9\u05D9\u05DD"
      },
      headerTitle__account: "\u05D7\u05E9\u05D1\u05D5\u05DF",
      headerTitle__security: "\u05D0\u05D1\u05D8\u05D7\u05D4",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "\u05E6\u05D5\u05E8 \u05E7\u05D5\u05D3\u05D9\u05DD \u05DE\u05D7\u05D3\u05E9",
          headerTitle: "\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
          subtitle__regenerate: "\u05E7\u05D1\u05DC \u05E7\u05D1\u05D5\u05E6\u05D4 \u05D7\u05D3\u05E9\u05D4 \u05E9\u05DC \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05DE\u05D0\u05D5\u05D1\u05D8\u05D7\u05D9\u05DD. \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05E7\u05D5\u05D3\u05DE\u05D9\u05DD \u05D9\u05D9\u05DE\u05D7\u05E7\u05D5 \u05D5\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D4\u05DD.",
          title__regenerate: "\u05E6\u05D5\u05E8 \u05DE\u05D7\u05D3\u05E9 \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9"
        },
        phoneCode: {
          actionLabel__setDefault: "\u05D4\u05D2\u05D3\u05E8 \u05DB\u05D1\u05E8\u05D9\u05E8\u05EA \u05DE\u05D7\u05D3\u05DC",
          destructiveActionLabel: "\u05D4\u05E1\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF"
        },
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9",
        title: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9",
        totp: {
          destructiveActionTitle: "\u05D4\u05E1\u05E8",
          headerTitle: "\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05DE\u05D0\u05DE\u05EA"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "\u05D4\u05D2\u05D3\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D4",
        primaryButton__updatePassword: "\u05E9\u05E0\u05D4 \u05E1\u05D9\u05E1\u05DE\u05D4",
        title: "\u05E1\u05D9\u05E1\u05DE\u05D4"
      },
      phoneNumbersSection: {
        destructiveAction: "\u05D4\u05E1\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
        detailsAction__nonPrimary: "\u05D4\u05D2\u05D3\u05E8 \u05DB\u05E8\u05D0\u05E9\u05D9",
        detailsAction__primary: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        detailsAction__unverified: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
        title: "\u05DE\u05E1\u05E4\u05E8\u05D9 \u05D8\u05DC\u05E4\u05D5\u05DF"
      },
      profileSection: {
        primaryButton: "",
        title: "\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC"
      },
      usernameSection: {
        primaryButton__setUsername: "\u05D4\u05D2\u05D3\u05E8 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
        primaryButton__updateUsername: "\u05E9\u05E0\u05D4 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
        title: "\u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9"
      },
      web3WalletsSection: {
        destructiveAction: "\u05D4\u05E1\u05E8 \u05D0\u05E8\u05E0\u05E7",
        primaryButton: "\u05D0\u05E8\u05E0\u05E7\u05D9 Web3",
        title: "\u05D0\u05E8\u05E0\u05E7\u05D9 Web3"
      }
    },
    usernamePage: {
      successMessage: "\u05E9\u05DD \u05D4\u05DE\u05E9\u05EA\u05DE\u05E9 \u05E9\u05DC\u05DA \u05E2\u05D5\u05D3\u05DB\u05DF.",
      title__set: "\u05E2\u05D3\u05DB\u05DF \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
      title__update: "\u05E2\u05D3\u05DB\u05DF \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05D0\u05E8\u05E0\u05E7 web3 \u05D6\u05D4.",
        successMessage: "{{web3Wallet}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05D0\u05E8\u05E0\u05E7 web3"
      },
      subtitle__availableWallets: "\u05D1\u05D7\u05E8 \u05D0\u05E8\u05E0\u05E7 web3 \u05DC\u05D7\u05D9\u05D1\u05D5\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      subtitle__unavailableWallets: "\u05D0\u05D9\u05DF \u05D0\u05E8\u05E0\u05E7\u05D9 web3 \u05D6\u05DE\u05D9\u05E0\u05D9\u05DD.",
      successMessage: "\u05D4\u05D0\u05E8\u05E0\u05E7 \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05E8\u05E0\u05E7 web3"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  heIL
});
