// @clerk/localizations/dist/sv-SE.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/sv-SE.ts
var sv_SE_exports = {};
__export(sv_SE_exports, {
  svSE: () => svSE
});
module.exports = __toCommonJS(sv_SE_exports);
var svSE = {
  locale: "sv-SE",
  backButton: "Tillbaka",
  badge__default: "Standard",
  badge__otherImpersonatorDevice: "Annans imitat\xF6renhet",
  badge__primary: "Prim\xE4r",
  badge__requiresAction: "Kr\xE4ver \xE5tg\xE4rd",
  badge__thisDevice: "Den h\xE4r enheten",
  badge__unverified: "Overifierad",
  badge__userDevice: "Anv\xE4ndarens enhet",
  badge__you: "Du",
  createOrganization: {
    formButtonSubmit: "Skapa organisation",
    invitePage: {
      formButtonReset: "Hoppa \xF6ver"
    },
    title: "Skapa organisation"
  },
  dates: {
    lastDay: "Ig\xE5r klockan {{ date | timeString('sv-SE') }}",
    next6Days: "{{ date | weekday('sv-SE','long') }} klockan {{ date | timeString('sv-SE') }}",
    nextDay: "Imorgon klockan {{ date | timeString('sv-SE') }}",
    numeric: "{{ date | numeric('sv-SE') }}",
    previous6Days: "Senaste {{ date | weekday('sv-SE','long') }} klockan {{ date | timeString('sv-SE') }}",
    sameDay: "Idag klockan {{ date | timeString('sv-SE') }}"
  },
  dividerText: "eller",
  footerActionLink__useAnotherMethod: "Anv\xE4nd en annan metod",
  footerPageLink__help: "Hj\xE4lp",
  footerPageLink__privacy: "Integritet",
  footerPageLink__terms: "Villkor",
  formButtonPrimary: "Forts\xE4tt",
  formButtonPrimary__verify: "Verify",
  formFieldAction__forgotPassword: "Gl\xF6mt l\xF6senord?",
  formFieldError__matchingPasswords: "Passwords match.",
  formFieldError__notMatchingPasswords: "Passwords don't match.",
  formFieldError__verificationLinkExpired: "The verification link expired. Please request a new link.",
  formFieldHintText__optional: "Valfritt",
  formFieldHintText__slug: "A slug is a human-readable ID that must be unique. It\u2019s often used in URLs.",
  formFieldInputPlaceholder__backupCode: "",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Delete account",
  formFieldInputPlaceholder__emailAddress: "",
  formFieldInputPlaceholder__emailAddress_username: "",
  formFieldInputPlaceholder__emailAddresses: "Ange eller klistra in en eller flera e-postadresser, separerade med mellanslag eller kommatecken",
  formFieldInputPlaceholder__firstName: "",
  formFieldInputPlaceholder__lastName: "",
  formFieldInputPlaceholder__organizationDomain: "",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "",
  formFieldInputPlaceholder__organizationName: "",
  formFieldInputPlaceholder__organizationSlug: "",
  formFieldInputPlaceholder__password: "",
  formFieldInputPlaceholder__phoneNumber: "",
  formFieldInputPlaceholder__username: "",
  formFieldLabel__automaticInvitations: "Enable automatic invitations for this domain",
  formFieldLabel__backupCode: "Reserv-kod",
  formFieldLabel__confirmDeletion: "Confirmation",
  formFieldLabel__confirmPassword: "Bekr\xE4fta l\xF6senord",
  formFieldLabel__currentPassword: "Nuvarande l\xF6senord",
  formFieldLabel__emailAddress: "E-postadress",
  formFieldLabel__emailAddress_username: "E-postadress eller anv\xE4ndarnamn",
  formFieldLabel__emailAddresses: "E-postadresser",
  formFieldLabel__firstName: "F\xF6rnamn",
  formFieldLabel__lastName: "Efternamn",
  formFieldLabel__newPassword: "Nytt l\xF6senord",
  formFieldLabel__organizationDomain: "Domain",
  formFieldLabel__organizationDomainDeletePending: "Delete pending invitations and suggestions",
  formFieldLabel__organizationDomainEmailAddress: "Verification email address",
  formFieldLabel__organizationDomainEmailAddressDescription: "Enter an email address under this domain to receive a code and verify this domain.",
  formFieldLabel__organizationName: "Organisationsnamn",
  formFieldLabel__organizationSlug: "Slug",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "L\xF6senord",
  formFieldLabel__phoneNumber: "Telefonnummer",
  formFieldLabel__role: "Roll",
  formFieldLabel__signOutOfOtherSessions: "Logga ut fr\xE5n alla andra enheter",
  formFieldLabel__username: "Anv\xE4ndarnamn",
  impersonationFab: {
    action__signOut: "Logga ut",
    title: "Inloggad som {{identifier}}"
  },
  maintenanceMode: void 0,
  membershipRole__admin: "Admin",
  membershipRole__basicMember: "Medlem",
  membershipRole__guestMember: "G\xE4st",
  organizationList: {
    action__createOrganization: "Create organization",
    action__invitationAccept: "Join",
    action__suggestionsAccept: "Request to join",
    createOrganization: "Create Organization",
    invitationAcceptedLabel: "Joined",
    subtitle: "to continue to {{applicationName}}",
    suggestionsAcceptedLabel: "Pending approval",
    title: "Choose an account",
    titleWithoutPersonal: "Choose an organization"
  },
  organizationProfile: {
    badge__automaticInvitation: "Automatic invitations",
    badge__automaticSuggestion: "Automatic suggestions",
    badge__manualInvitation: "No automatic enrollment",
    badge__unverified: "Unverified",
    createDomainPage: {
      subtitle: "Add the domain to verify. Users with email addresses at this domain can join the organization automatically or request to join.",
      title: "Add domain"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Inbjudningarna kunde inte skickas. \xC5tg\xE4rda f\xF6ljande och f\xF6rs\xF6k igen:",
      formButtonPrimary__continue: "Skicka inbjudningar",
      selectDropdown__role: "Select role",
      subtitle: "Bjud in nya medlemmar till denna organisation",
      successMessage: "Inbjudningar skickade",
      title: "Bjud in medlemmar"
    },
    membersPage: {
      action__invite: "Bjud in",
      activeMembersTab: {
        menuAction__remove: "Ta bort medlem",
        tableHeader__actions: "",
        tableHeader__joined: "Gick med",
        tableHeader__role: "Roll",
        tableHeader__user: "Anv\xE4ndare"
      },
      detailsTitle__emptyRow: "Inga medlemmar att visa",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invite users by connecting an email domain with your organization. Anyone who signs up with a matching email domain will be able to join the organization anytime.",
          headerTitle: "Automatic invitations",
          primaryButton: "Manage verified domains"
        },
        table__emptyRow: "No invitations to display"
      },
      invitedMembersTab: {
        menuAction__revoke: "\xC5terkalla inbjudan",
        tableHeader__invited: "Inbjudna"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Users who sign up with a matching email domain, will be able to see a suggestion to request to join your organization.",
          headerTitle: "Automatic suggestions",
          primaryButton: "Manage verified domains"
        },
        menuAction__approve: "Approve",
        menuAction__reject: "Reject",
        tableHeader__requested: "Requested access",
        table__emptyRow: "No requests to display"
      },
      start: {
        headerTitle__invitations: "Invitations",
        headerTitle__members: "Members",
        headerTitle__requests: "Requests"
      }
    },
    navbar: {
      description: "Manage your organization.",
      general: "General",
      members: "Members",
      title: "Organization"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Are you sure you want to delete this organization?",
          messageLine2: "This action is permanent and irreversible.",
          successMessage: "You have deleted the organization.",
          title: "Delete organization"
        },
        leaveOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "\xC4r du s\xE4ker p\xE5 att du vill l\xE4mna denna organisation? Du kommer att f\xF6rlora \xE5tkomst till organisationen och dess applikationer.",
          messageLine2: "Denna \xE5tg\xE4rd \xE4r permanent och o\xE5terkallelig.",
          successMessage: "Du har l\xE4mnat organisationen.",
          title: "L\xE4mna organisation"
        },
        title: "Farligt"
      },
      domainSection: {
        menuAction__manage: "Manage",
        menuAction__remove: "Delete",
        menuAction__verify: "Verify",
        primaryButton: "Add domain",
        subtitle: "Allow users to join the organization automatically or request to join based on a verified email domain.",
        title: "Verified domains"
      },
      successMessage: "Organisationen har uppdaterats.",
      title: "Organisationsprofil"
    },
    removeDomainPage: {
      messageLine1: "The email domain {{domain}} will be removed.",
      messageLine2: "Users won\u2019t be able to join the organization automatically after this.",
      successMessage: "{{domain}} has been removed.",
      title: "Remove domain"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "Medlemmar",
      profileSection: {
        primaryButton: "",
        title: "Organization Profile",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Removing this domain will affect invited users.",
        removeDomainActionLabel__remove: "Remove domain",
        removeDomainSubtitle: "Remove this domain from your verified domains",
        removeDomainTitle: "Remove domain"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Users are automatically invited to join the organization when they sign-up and can join anytime.",
        automaticInvitationOption__label: "Automatic invitations",
        automaticSuggestionOption__description: "Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.",
        automaticSuggestionOption__label: "Automatic suggestions",
        calloutInfoLabel: "Changing the enrollment mode will only affect new users.",
        calloutInvitationCountLabel: "Pending invitations sent to users: {{count}}",
        calloutSuggestionCountLabel: "Pending suggestions sent to users: {{count}}",
        manualInvitationOption__description: "Users can only be invited manually to the organization.",
        manualInvitationOption__label: "No automatic enrollment",
        subtitle: "Choose how users from this domain can join the organization."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Enrollment options"
      },
      subtitle: "The domain {{domain}} is now verified. Continue by selecting enrollment mode.",
      title: "Update {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Enter the verification code sent to your email address",
      formTitle: "Verification code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "The domain {{domainName}} needs to be verified via email.",
      subtitleVerificationCodeScreen: "A verification code was sent to {{emailAddress}}. Enter the code to continue.",
      title: "Verify domain"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "Skapa organisation",
    action__invitationAccept: "Join",
    action__manageOrganization: "Hantera organisation",
    action__suggestionsAccept: "Request to join",
    notSelected: "Ingen organisation vald",
    personalWorkspace: "Personligt Arbetsomr\xE5de",
    suggestionsAcceptedLabel: "Pending approval"
  },
  paginationButton__next: "N\xE4sta",
  paginationButton__previous: "F\xF6reg\xE5ende",
  paginationRowText__displaying: "Visar",
  paginationRowText__of: "av",
  signIn: {
    accountSwitcher: {
      action__addAccount: "Add account",
      action__signOutAll: "Sign out of all accounts",
      subtitle: "Select the account with which you wish to continue.",
      title: "Choose an account"
    },
    alternativeMethods: {
      actionLink: "F\xE5 hj\xE4lp",
      actionText: "Don\u2019t have any of these?",
      blockButton__backupCode: "Anv\xE4nd en reservkod",
      blockButton__emailCode: "Skicka kod till {{identifier}}",
      blockButton__emailLink: "Skicka l\xE4nk till {{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "Logga in med ditt l\xF6senord",
      blockButton__phoneCode: "Skicka kod till {{identifier}}",
      blockButton__totp: "Anv\xE4nd din autentiseringsapp",
      getHelp: {
        blockButton__emailSupport: "E-post support",
        content: "Om du har problem med att logga in p\xE5 ditt konto, kontakta oss via e-post s\xE5 hj\xE4lper vi dig att \xE5terst\xE4lla \xE5tkomsten s\xE5 snabbt som m\xF6jligt.",
        title: "F\xE5 hj\xE4lp"
      },
      subtitle: "Facing issues? You can use any of these methods to sign in.",
      title: "Anv\xE4nd en annan metod"
    },
    backupCodeMfa: {
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Ange en reservkod"
    },
    emailCode: {
      formTitle: "Verifieringskod",
      resendButton: "Skicka koden igen",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Kontrollera din e-post"
    },
    emailLink: {
      expired: {
        subtitle: "\xC5terg\xE5 till ursprungliga fliken f\xF6r att forts\xE4tta.",
        title: "Denna verifieringsl\xE4nk har upph\xF6rt att g\xE4lla"
      },
      failed: {
        subtitle: "\xC5terg\xE5 till ursprungliga fliken f\xF6r att forts\xE4tta.",
        title: "Denna verifieringsl\xE4nk \xE4r ogiltig"
      },
      formSubtitle: "Anv\xE4nd verifieringsl\xE4nken som skickades till din e-postadress",
      formTitle: "Verifieringsl\xE4nk",
      loading: {
        subtitle: "Du kommer att omdirigeras snart",
        title: "Loggar in..."
      },
      resendButton: "Skicka l\xE4nken igen",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Kontrollera din e-post",
      unusedTab: {
        title: "Du kan st\xE4nga den h\xE4r fliken"
      },
      verified: {
        subtitle: "Du kommer att omdirigeras snart",
        title: "Inloggningen lyckades"
      },
      verifiedSwitchTab: {
        subtitle: "\xC5terg\xE5 till ursprungliga fliken f\xF6r att forts\xE4tta",
        subtitleNewTab: "\xC5terg\xE5 till den nyligen \xF6ppnade fliken f\xF6r att forts\xE4tta",
        titleNewTab: "Loggade in p\xE5 annan flik"
      }
    },
    forgotPassword: {
      formTitle: "Reset password code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "to reset your password",
      subtitle_email: "First, enter the code sent to your email ID",
      subtitle_phone: "First, enter the code sent to your phone",
      title: "Reset password"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "Reset your password",
      label__alternativeMethods: "Or, sign in with another method",
      title: "Forgot Password?"
    },
    noAvailableMethods: {
      message: "Kan inte forts\xE4tta med inloggning. Det finns ingen tillg\xE4nglig autentiseringsfaktor.",
      subtitle: "Ett fel intr\xE4ffade",
      title: "Kan inte logga in"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "Anv\xE4nd en annan metod",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Ange ditt l\xF6senord"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "Verifieringskod",
      resendButton: "Skicka koden igen",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Kolla din telefon"
    },
    phoneCodeMfa: {
      formTitle: "Verifieringskod",
      resendButton: "Skicka koden igen",
      subtitle: "",
      title: "Kolla din telefon"
    },
    resetPassword: {
      formButtonPrimary: "Reset Password",
      requiredMessage: "For security reasons, it is required to reset your password.",
      successMessage: "Your password was successfully changed. Signing you in, please wait a moment.",
      title: "Set new password"
    },
    resetPasswordMfa: {
      detailsLabel: "We need to verify your identity before resetting your password."
    },
    start: {
      actionLink: "Skapa konto",
      actionLink__use_email: "Use email",
      actionLink__use_email_username: "Use email or username",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "Use phone",
      actionLink__use_username: "Use username",
      actionText: "Har du inget konto?",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Logga in"
    },
    totpMfa: {
      formTitle: "Verifieringskod",
      subtitle: "",
      title: "Tv\xE5stegsverifiering"
    }
  },
  signInEnterPasswordTitle: "Ange ditt l\xF6senord",
  signUp: {
    continue: {
      actionLink: "Logga in",
      actionText: "Har du redan ett konto?",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Fyll i n\xF6dv\xE4ndiga f\xE4lt"
    },
    emailCode: {
      formSubtitle: "Ange verifieringskoden som skickades till din e-postadress",
      formTitle: "Verifieringskod",
      resendButton: "Skicka koden igen",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Verifiera din e-post"
    },
    emailLink: {
      formSubtitle: "Anv\xE4nd verifieringsl\xE4nken som skickades till din e-postadress",
      formTitle: "Verifieringsl\xE4nk",
      loading: {
        title: "Registrerar..."
      },
      resendButton: "Skicka l\xE4nken igen",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Verifiera din e-post",
      verified: {
        title: "Registreringen lyckades"
      },
      verifiedSwitchTab: {
        subtitle: "\xC5terg\xE5 till den nyligen \xF6ppnade fliken f\xF6r att forts\xE4tta",
        subtitleNewTab: "\xC5terg\xE5 till f\xF6reg\xE5ende flik f\xF6r att forts\xE4tta",
        title: "E-posten har verifierats"
      }
    },
    phoneCode: {
      formSubtitle: "Ange verifieringskoden som skickades till ditt telefonnummer",
      formTitle: "Verifieringskod",
      resendButton: "Skicka koden igen",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Verifiera din telefon"
    },
    start: {
      actionLink: "Logga in",
      actionText: "Har du redan ett konto?",
      subtitle: "f\xF6r att forts\xE4tta till {{applicationName}}",
      title: "Skapa ditt konto"
    }
  },
  socialButtonsBlockButton: "Forts\xE4tt med {{provider|titleize}}",
  unstable__errors: {
    captcha_invalid: "Sign up unsuccessful due to failed security validations. Please refresh the page to try again or reach out to support for more assistance.",
    captcha_unavailable: "Sign up unsuccessful due to failed bot validation. Please refresh the page to try again or reach out to support for more assistance.",
    form_code_incorrect: "",
    form_identifier_exists: "",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "",
    form_param_format_invalid: "",
    form_param_format_invalid__email_address: "Email address must be a valid email address.",
    form_param_format_invalid__phone_number: "Phone number must be in a valid international format",
    form_param_max_length_exceeded__first_name: "First name should not exceed 256 characters.",
    form_param_max_length_exceeded__last_name: "Last name should not exceed 256 characters.",
    form_param_max_length_exceeded__name: "Name should not exceed 256 characters.",
    form_param_nil: "",
    form_password_incorrect: "",
    form_password_length_too_short: "",
    form_password_not_strong_enough: "Your password is not strong enough.",
    form_password_pwned: "",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "Your password has exceeded the maximum number of bytes allowed, please shorten it or remove some special characters.",
    form_password_validation_failed: "Incorrect Password",
    form_username_invalid_character: "",
    form_username_invalid_length: "",
    identification_deletion_failed: "You cannot delete your last identification.",
    not_allowed_access: "",
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "",
      minimumLength: "",
      requireLowercase: "",
      requireNumbers: "",
      requireSpecialCharacter: "",
      requireUppercase: "",
      sentencePrefix: ""
    },
    phone_number_exists: "This phone number is taken. Please try another.",
    zxcvbn: {
      couldBeStronger: "Your password works, but could be stronger. Try adding more characters.",
      goodPassword: "Your password meets all the necessary requirements.",
      notEnough: "Your password is not strong enough.",
      suggestions: {
        allUppercase: "Capitalize some, but not all letters.",
        anotherWord: "Add more words that are less common.",
        associatedYears: "Avoid years that are associated with you.",
        capitalization: "Capitalize more than the first letter.",
        dates: "Avoid dates and years that are associated with you.",
        l33t: "Avoid predictable letter substitutions like '@' for 'a'.",
        longerKeyboardPattern: "Use longer keyboard patterns and change typing direction multiple times.",
        noNeed: "You can create strong passwords without using symbols, numbers, or uppercase letters.",
        pwned: "If you use this password elsewhere, you should change it.",
        recentYears: "Avoid recent years.",
        repeated: "Avoid repeated words and characters.",
        reverseWords: "Avoid reversed spellings of common words.",
        sequences: "Avoid common character sequences.",
        useWords: "Use multiple words, but avoid common phrases."
      },
      warnings: {
        common: "This is a commonly used password.",
        commonNames: "Common names and surnames are easy to guess.",
        dates: "Dates are easy to guess.",
        extendedRepeat: 'Repeated character patterns like "abcabcabc" are easy to guess.',
        keyPattern: "Short keyboard patterns are easy to guess.",
        namesByThemselves: "Single names or surnames are easy to guess.",
        pwned: "Your password was exposed by a data breach on the Internet.",
        recentYears: "Recent years are easy to guess.",
        sequences: 'Common character sequences like "abc" are easy to guess.',
        similarToCommon: "This is similar to a commonly used password.",
        simpleRepeat: 'Repeated characters like "aaa" are easy to guess.',
        straightRow: "Straight rows of keys on your keyboard are easy to guess.",
        topHundred: "This is a frequently used password.",
        topTen: "This is a heavily used password.",
        userInputs: "There should not be any personal or page related data.",
        wordByItself: "Single words are easy to guess."
      }
    }
  },
  userButton: {
    action__addAccount: "L\xE4gg till konto",
    action__manageAccount: "Hantera konto",
    action__signOut: "Logga ut",
    action__signOutAll: "Logga ut fr\xE5n alla konton"
  },
  userProfile: {
    backupCodePage: {
      actionLabel__copied: "Kopierat!",
      actionLabel__copy: "Kopiera alla",
      actionLabel__download: "Ladda ner .txt",
      actionLabel__print: "Skriv ut",
      infoText1: "Backupkoder kommer att aktiveras f\xF6r detta konto.",
      infoText2: "H\xE5ll backupkoderna hemliga och f\xF6rvara dem s\xE4kert. Du kan generera nya backupkoder om du misst\xE4nker att de har komprometterats.",
      subtitle__codelist: "F\xF6rvara dem s\xE4kert och h\xE5ll dem hemliga.",
      successMessage: "Backupkoder \xE4r nu aktiverade. Du kan anv\xE4nda en av dessa f\xF6r att logga in p\xE5 ditt konto om du f\xF6rlorar \xE5tkomsten till din autentiseringsenhet. Varje kod kan endast anv\xE4ndas en g\xE5ng.",
      successSubtitle: "Du kan anv\xE4nda en av dessa f\xF6r att logga in p\xE5 ditt konto om du f\xF6rlorar \xE5tkomsten till din autentiseringsenhet.",
      title: "L\xE4gg till backupkodverifiering",
      title__codelist: "Backupkoder"
    },
    connectedAccountPage: {
      formHint: "V\xE4lj en leverant\xF6r f\xF6r att ansluta ditt konto.",
      formHint__noAccounts: "Det finns inga tillg\xE4ngliga externa kontoleverant\xF6rer.",
      removeResource: {
        messageLine1: "{{identifier}} kommer att tas bort fr\xE5n detta konto.",
        messageLine2: "Du kommer inte l\xE4ngre att kunna anv\xE4nda detta anslutna konto och alla beroende funktioner kommer att sluta fungera.",
        successMessage: "{{connectedAccount}} har tagits bort fr\xE5n ditt konto.",
        title: "Ta bort anslutet konto"
      },
      socialButtonsBlockButton: "Anslut {{provider|titleize}} konto",
      successMessage: "Leverant\xF6ren har lagts till i ditt konto.",
      title: "L\xE4gg till anslutet konto"
    },
    deletePage: {
      actionDescription: 'Type "Delete account" below to continue.',
      confirm: "Delete account",
      messageLine1: "Are you sure you want to delete your account?",
      messageLine2: "This action is permanent and irreversible.",
      title: "Delete account"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "Ett e-postmeddelande med en verifieringskod kommer att skickas till denna e-postadress.",
        formSubtitle: "Ange verifieringskoden som skickats till {{identifier}}",
        formTitle: "Verifieringskod",
        resendButton: "Skicka kod igen",
        successMessage: "E-postadressen {{identifier}} har lagts till i ditt konto."
      },
      emailLink: {
        formHint: "Ett e-postmeddelande med en verifieringsl\xE4nk kommer att skickas till denna e-postadress.",
        formSubtitle: "Klicka p\xE5 verifieringsl\xE4nken i e-postmeddelandet som skickats till {{identifier}}",
        formTitle: "Verifieringsl\xE4nk",
        resendButton: "Skicka l\xE4nken igen",
        successMessage: "E-postadressen {{identifier}} har lagts till i ditt konto."
      },
      removeResource: {
        messageLine1: "{{identifier}} kommer att tas bort fr\xE5n detta konto.",
        messageLine2: "Du kommer inte l\xE4ngre att kunna logga in med denna e-postadress.",
        successMessage: "{{emailAddress}} har tagits bort fr\xE5n ditt konto.",
        title: "Ta bort e-postadress"
      },
      title: "L\xE4gg till e-postadress",
      verifyTitle: "Verify email address"
    },
    formButtonPrimary__add: "Add",
    formButtonPrimary__continue: "Forts\xE4tt",
    formButtonPrimary__finish: "Slutf\xF6r",
    formButtonPrimary__remove: "Remove",
    formButtonPrimary__save: "Save",
    formButtonReset: "Avbryt",
    mfaPage: {
      formHint: "V\xE4lj en metod att l\xE4gga till.",
      title: "L\xE4gg till tv\xE5stegsverifiering"
    },
    mfaPhoneCodePage: {
      backButton: "Use existing number",
      primaryButton__addPhoneNumber: "L\xE4gg till ett telefonnummer",
      removeResource: {
        messageLine1: "{{identifier}} kommer inte l\xE4ngre att ta emot verifieringskoder vid inloggning.",
        messageLine2: "Ditt konto kan vara mindre s\xE4kert. \xC4r du s\xE4ker p\xE5 att du vill forts\xE4tta?",
        successMessage: "SMS-kod tv\xE5stegsverifiering har tagits bort f\xF6r {{mfaPhoneCode}}",
        title: "Ta bort tv\xE5stegsverifiering"
      },
      subtitle__availablePhoneNumbers: "V\xE4lj ett telefonnummer att registrera f\xF6r SMS-kod tv\xE5stegsverifiering.",
      subtitle__unavailablePhoneNumbers: "Det finns inga tillg\xE4ngliga telefonnummer att registrera f\xF6r SMS-kod tv\xE5stegsverifiering.",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS code verification enabled",
      title: "L\xE4gg till SMS-kodverifiering"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Skanna QR-kod ist\xE4llet",
        buttonUnableToScan__nonPrimary: "Kan inte skanna QR-kod?",
        infoText__ableToScan: "Konfigurera en ny inloggningsmetod i din autentiseringsapp och skanna f\xF6ljande QR-kod f\xF6r att l\xE4nka den till ditt konto.",
        infoText__unableToScan: "Konfigurera en ny inloggningsmetod i din autentiseringsapp och ange nyckeln nedan.",
        inputLabel__unableToScan1: "Se till att tidsbaserade eller eng\xE5ngsl\xF6senord \xE4r aktiverade och slutf\xF6r sedan l\xE4nkningen till ditt konto.",
        inputLabel__unableToScan2: "Alternativt, om din autentiseringsapp st\xF6djer TOTP URI kan du ocks\xE5 kopiera hela URI."
      },
      removeResource: {
        messageLine1: "Verifieringskoder fr\xE5n denna autentiseringsapp kommer inte l\xE4ngre att kr\xE4vas vid inloggning.",
        messageLine2: "Ditt konto kan vara mindre s\xE4kert. \xC4r du s\xE4ker p\xE5 att du vill forts\xE4tta?",
        successMessage: "Tv\xE5stegsverifiering via autentiseringsapp har tagits bort.",
        title: "Ta bort tv\xE5stegsverifiering"
      },
      successMessage: "Tv\xE5stegsverifiering \xE4r nu aktiverat. Vid inloggning beh\xF6ver du ange en verifieringskod fr\xE5n denna autentiseringsapp som ett extra steg.",
      title: "L\xE4gg till autentiseringsapp",
      verifySubtitle: "Ange verifieringskoden genererad av din autentiseringsapp",
      verifyTitle: "Verifieringskod"
    },
    mobileButton__menu: "Meny",
    navbar: {
      account: "Profile",
      description: "Manage your account info.",
      security: "Security",
      title: "Account"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "It is recommended to sign out of all other devices which may have used your old password.",
      readonly: "Your password can currently not be edited because you can sign in only via the enterprise connection.",
      successMessage__set: "Ditt l\xF6senord har angetts.",
      successMessage__signOutOfOtherSessions: "Alla andra enheter har loggats ut.",
      successMessage__update: "Ditt l\xF6senord har uppdaterats.",
      title__set: "Ange l\xF6senord",
      title__update: "Byt l\xF6senord"
    },
    phoneNumberPage: {
      infoText: "Ett textmeddelande med en verifieringsl\xE4nk kommer att skickas till detta telefonnummer.",
      removeResource: {
        messageLine1: "{{identifier}} kommer att tas bort fr\xE5n detta konto.",
        messageLine2: "Du kommer inte l\xE4ngre att kunna logga in med detta telefonnummer.",
        successMessage: "{{phoneNumber}} har tagits bort fr\xE5n ditt konto.",
        title: "Ta bort telefonnummer"
      },
      successMessage: "{{identifier}} har lagts till i ditt konto.",
      title: "L\xE4gg till telefonnummer",
      verifySubtitle: "Enter the verification code sent to {{identifier}}",
      verifyTitle: "Verify phone number"
    },
    profilePage: {
      fileDropAreaHint: "Ladda upp en JPG, PNG, GIF, eller WEBP bild som \xE4r mindre \xE4n 10 MB",
      imageFormDestructiveActionSubtitle: "Ta bort bild",
      imageFormSubtitle: "Ladda upp bild",
      imageFormTitle: "Profilbild",
      readonly: "Your profile information has been provided by the enterprise connection and cannot be edited.",
      successMessage: "Din profil har uppdaterats.",
      title: "Uppdatera profil"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Logga ut fr\xE5n enhet",
        title: "Aktiva enheter"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "F\xF6rs\xF6k igen",
        actionLabel__reauthorize: "Autorisera nu",
        destructiveActionTitle: "Ta bort",
        primaryButton: "Anslut konto",
        subtitle__reauthorize: "The required scopes have been updated, and you may be experiencing limited functionality. Please re-authorize this application to avoid any issues",
        title: "Anslutna konton"
      },
      dangerSection: {
        deleteAccountButton: "Delete Account",
        title: "Account termination"
      },
      emailAddressesSection: {
        destructiveAction: "Ta bort e-postadress",
        detailsAction__nonPrimary: "S\xE4tt som prim\xE4r",
        detailsAction__primary: "Fullborda verifiering",
        detailsAction__unverified: "Fullborda verifiering",
        primaryButton: "L\xE4gg till en e-postadress",
        title: "E-postadresser"
      },
      enterpriseAccountsSection: {
        title: "Enterprise accounts"
      },
      headerTitle__account: "Konto",
      headerTitle__security: "S\xE4kerhet",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "\xC5tergenerera koder",
          headerTitle: "S\xE4kerhetskopieringskoder",
          subtitle__regenerate: "F\xE5 en ny upps\xE4ttning s\xE4kra s\xE4kerhetskopieringskoder. Tidigare koder kommer att raderas och kan inte anv\xE4ndas.",
          title__regenerate: "\xC5tergenerera s\xE4kerhetskopieringskoder"
        },
        phoneCode: {
          actionLabel__setDefault: "Ange som standard",
          destructiveActionLabel: "Ta bort telefonnummer"
        },
        primaryButton: "L\xE4gg till tv\xE5stegsverifiering",
        title: "Tv\xE5stegsverifiering",
        totp: {
          destructiveActionTitle: "Ta bort",
          headerTitle: "Autentiseringsapp"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "St\xE4ll in l\xF6senord",
        primaryButton__updatePassword: "Byt l\xF6senord",
        title: "L\xF6senord"
      },
      phoneNumbersSection: {
        destructiveAction: "Ta bort telefonnummer",
        detailsAction__nonPrimary: "S\xE4tt som prim\xE4r",
        detailsAction__primary: "Fullborda verifiering",
        detailsAction__unverified: "Fullborda verifiering",
        primaryButton: "L\xE4gg till ett telefonnummer",
        title: "Telefonnummer"
      },
      profileSection: {
        primaryButton: "",
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "S\xE4tt anv\xE4ndarnamn",
        primaryButton__updateUsername: "\xC4ndra anv\xE4ndarnamn",
        title: "Anv\xE4ndarnamn"
      },
      web3WalletsSection: {
        destructiveAction: "Ta bort pl\xE5nbok",
        primaryButton: "Web3 pl\xE5nb\xF6cker",
        title: "Web3 pl\xE5nb\xF6cker"
      }
    },
    usernamePage: {
      successMessage: "Ditt anv\xE4ndarnamn har uppdaterats.",
      title__set: "Uppdatera anv\xE4ndarnamn",
      title__update: "Uppdatera anv\xE4ndarnamn"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} kommer att tas bort fr\xE5n detta konto.",
        messageLine2: "Du kommer inte l\xE4ngre att kunna logga in med denna web3-pl\xE5nbok.",
        successMessage: "{{web3Wallet}} har tagits bort fr\xE5n ditt konto.",
        title: "Ta bort web3-pl\xE5nbok"
      },
      subtitle__availableWallets: "V\xE4lj en web3-pl\xE5nbok att ansluta till ditt konto.",
      subtitle__unavailableWallets: "Det finns inga tillg\xE4ngliga web3-pl\xE5nb\xF6cker.",
      successMessage: "Pl\xE5nboken har lagts till i ditt konto.",
      title: "L\xE4gg till web3-pl\xE5nbok"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  svSE
});
