// @clerk/localizations/dist/tr-TR.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/tr-TR.ts
var tr_TR_exports = {};
__export(tr_TR_exports, {
  trTR: () => trTR
});
module.exports = __toCommonJS(tr_TR_exports);
var trTR = {
  locale: "tr-TR",
  backButton: "Geri",
  badge__default: "Varsay\u0131lan",
  badge__otherImpersonatorDevice: "Farkl\u0131 bir k\u0131l\u0131k\xE7\u0131 cihaz\u0131",
  badge__primary: "Birincil",
  badge__requiresAction: "Eylem gerekli",
  badge__thisDevice: "Bu cihaz",
  badge__unverified: "Do\u011Frulanmam\u0131\u015F",
  badge__userDevice: "Kullan\u0131c\u0131 cihaz\u0131",
  badge__you: "Siz",
  createOrganization: {
    formButtonSubmit: "Olu\u015Ftur",
    invitePage: {
      formButtonReset: "Atla"
    },
    title: "Organizasyon olu\u015Ftur"
  },
  dates: {
    lastDay: "D\xFCn saat {{ date | timeString('tr-TR') }}",
    next6Days: "{{ date | weekday('tr-TR','long') }} saat {{ date | timeString('tr-TR') }}",
    nextDay: "Yar\u0131n saat {{ date | timeString('tr-TR') }}",
    numeric: "{{ date | numeric('tr-TR') }}",
    previous6Days: "Ge\xE7en hafta {{ date | weekday('tr-TR','long') }} saat {{ date | timeString('tr-TR') }}",
    sameDay: "Bug\xFCn saat {{ date | timeString('tr-TR') }}"
  },
  dividerText: "veya",
  footerActionLink__useAnotherMethod: "Ba\u015Fka bir y\xF6ntem kullan",
  footerPageLink__help: "Yard\u0131m",
  footerPageLink__privacy: "Gizlilik",
  footerPageLink__terms: "\u015Eartlar",
  formButtonPrimary: "\u0130leri",
  formButtonPrimary__verify: "Verify",
  formFieldAction__forgotPassword: "\u015Eifremi unuttum",
  formFieldError__matchingPasswords: "\u015Eifreler e\u015Fle\u015Fiyor.",
  formFieldError__notMatchingPasswords: "\u015Eifreler e\u015Fle\u015Fmiyor.",
  formFieldError__verificationLinkExpired: "The verification link expired. Please request a new link.",
  formFieldHintText__optional: "\u0130ste\u011Fe ba\u011Fl\u0131",
  formFieldHintText__slug: "A slug is a human-readable ID that must be unique. It\u2019s often used in URLs.",
  formFieldInputPlaceholder__backupCode: "",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Delete account",
  formFieldInputPlaceholder__emailAddress: "",
  formFieldInputPlaceholder__emailAddress_username: "",
  formFieldInputPlaceholder__emailAddresses: "Bo\u015Fluklar veya virg\xFCllerle ay\u0131rarak bir veya daha fazla e-posta adresi girin veya yap\u0131\u015Ft\u0131r\u0131n",
  formFieldInputPlaceholder__firstName: "",
  formFieldInputPlaceholder__lastName: "",
  formFieldInputPlaceholder__organizationDomain: "",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "",
  formFieldInputPlaceholder__organizationName: "",
  formFieldInputPlaceholder__organizationSlug: "",
  formFieldInputPlaceholder__password: "",
  formFieldInputPlaceholder__phoneNumber: "",
  formFieldInputPlaceholder__username: "",
  formFieldLabel__automaticInvitations: "Enable automatic invitations for this domain",
  formFieldLabel__backupCode: "Yedekleme kodu",
  formFieldLabel__confirmDeletion: "Confirmation",
  formFieldLabel__confirmPassword: "\u015Eifreyi onayla",
  formFieldLabel__currentPassword: "Mevcut \u015Fifre",
  formFieldLabel__emailAddress: "E-posta adresi",
  formFieldLabel__emailAddress_username: "E-posta adresi veya kullan\u0131c\u0131 ad\u0131",
  formFieldLabel__emailAddresses: "E-posta adresleri",
  formFieldLabel__firstName: "Ad",
  formFieldLabel__lastName: "Soyad",
  formFieldLabel__newPassword: "Yeni \u015Fifre",
  formFieldLabel__organizationDomain: "Domain",
  formFieldLabel__organizationDomainDeletePending: "Delete pending invitations and suggestions",
  formFieldLabel__organizationDomainEmailAddress: "Verification email address",
  formFieldLabel__organizationDomainEmailAddressDescription: "Enter an email address under this domain to receive a code and verify this domain.",
  formFieldLabel__organizationName: "Organizasyon ad\u0131",
  formFieldLabel__organizationSlug: "Organizasyon slug'\u0131",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "\u015Eifre",
  formFieldLabel__phoneNumber: "Telefon numaras\u0131",
  formFieldLabel__role: "Rol",
  formFieldLabel__signOutOfOtherSessions: "Di\u011Fer cihazlardaki oturumlardan \xE7\u0131k",
  formFieldLabel__username: "Kullan\u0131c\u0131 ad\u0131",
  impersonationFab: {
    action__signOut: "\xC7\u0131k\u0131\u015F yap",
    title: "{{identifier}} olarak giri\u015F yap\u0131ld\u0131"
  },
  maintenanceMode: void 0,
  membershipRole__admin: "Admin",
  membershipRole__basicMember: "\xDCye",
  membershipRole__guestMember: "Misafir",
  organizationList: {
    action__createOrganization: "Create organization",
    action__invitationAccept: "Join",
    action__suggestionsAccept: "Request to join",
    createOrganization: "Create Organization",
    invitationAcceptedLabel: "Joined",
    subtitle: "to continue to {{applicationName}}",
    suggestionsAcceptedLabel: "Pending approval",
    title: "Choose an account",
    titleWithoutPersonal: "Choose an organization"
  },
  organizationProfile: {
    badge__automaticInvitation: "Automatic invitations",
    badge__automaticSuggestion: "Automatic suggestions",
    badge__manualInvitation: "No automatic enrollment",
    badge__unverified: "Unverified",
    createDomainPage: {
      subtitle: "Add the domain to verify. Users with email addresses at this domain can join the organization automatically or request to join.",
      title: "Add domain"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Daveiyeler g\xF6nderilemedi. A\u015Fa\u011F\u0131da belirtilen sorunlar\u0131 giderip tekrar deneyin:",
      formButtonPrimary__continue: "Davetiye g\xF6nder",
      selectDropdown__role: "Select role",
      subtitle: "Bu organizasyona yeni \xFCyeler davet edin",
      successMessage: "Davetiyeler ba\u015Far\u0131yla g\xF6nderildi",
      title: "Davetiye g\xF6nder"
    },
    membersPage: {
      action__invite: "Davet et",
      activeMembersTab: {
        menuAction__remove: "\xDCyeyi kald\u0131r",
        tableHeader__actions: "",
        tableHeader__joined: "Kat\u0131lma tarihi",
        tableHeader__role: "Rol\xFC",
        tableHeader__user: "Kullan\u0131c\u0131"
      },
      detailsTitle__emptyRow: "G\xF6sterilecek \xFCye yok",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invite users by connecting an email domain with your organization. Anyone who signs up with a matching email domain will be able to join the organization anytime.",
          headerTitle: "Automatic invitations",
          primaryButton: "Manage verified domains"
        },
        table__emptyRow: "No invitations to display"
      },
      invitedMembersTab: {
        menuAction__revoke: "Davetiyeyi iptal et",
        tableHeader__invited: "Davetliler"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Users who sign up with a matching email domain, will be able to see a suggestion to request to join your organization.",
          headerTitle: "Automatic suggestions",
          primaryButton: "Manage verified domains"
        },
        menuAction__approve: "Approve",
        menuAction__reject: "Reject",
        tableHeader__requested: "Requested access",
        table__emptyRow: "No requests to display"
      },
      start: {
        headerTitle__invitations: "Invitations",
        headerTitle__members: "Members",
        headerTitle__requests: "Requests"
      }
    },
    navbar: {
      description: "Manage your organization.",
      general: "General",
      members: "Members",
      title: "Organization"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Are you sure you want to delete this organization?",
          messageLine2: "This action is permanent and irreversible.",
          successMessage: "You have deleted the organization.",
          title: "Delete organization"
        },
        leaveOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Bu organizasyondan ayr\u0131lmak istedi\u011Finizden emin misiniz? Bu organizasyona ve uygulamalar\u0131na eri\u015Fiminizi kaybedeceksiniz.",
          messageLine2: "bu eylem kal\u0131c\u0131d\u0131r ve geri al\u0131namaz.",
          successMessage: "Organizasyondan ayr\u0131ld\u0131n\u0131z.",
          title: "Organizasyondan ayr\u0131l"
        },
        title: "Tehlike"
      },
      domainSection: {
        menuAction__manage: "Manage",
        menuAction__remove: "Delete",
        menuAction__verify: "Verify",
        primaryButton: "Add domain",
        subtitle: "Allow users to join the organization automatically or request to join based on a verified email domain.",
        title: "Verified domains"
      },
      successMessage: "Organizasyon g\xFCncellendi.",
      title: "Organizasyon profili"
    },
    removeDomainPage: {
      messageLine1: "The email domain {{domain}} will be removed.",
      messageLine2: "Users won\u2019t be able to join the organization automatically after this.",
      successMessage: "{{domain}} has been removed.",
      title: "Remove domain"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "\xDCyeler",
      profileSection: {
        primaryButton: "",
        title: "Organization Profile",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Removing this domain will affect invited users.",
        removeDomainActionLabel__remove: "Remove domain",
        removeDomainSubtitle: "Remove this domain from your verified domains",
        removeDomainTitle: "Remove domain"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Users are automatically invited to join the organization when they sign-up and can join anytime.",
        automaticInvitationOption__label: "Automatic invitations",
        automaticSuggestionOption__description: "Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.",
        automaticSuggestionOption__label: "Automatic suggestions",
        calloutInfoLabel: "Changing the enrollment mode will only affect new users.",
        calloutInvitationCountLabel: "Pending invitations sent to users: {{count}}",
        calloutSuggestionCountLabel: "Pending suggestions sent to users: {{count}}",
        manualInvitationOption__description: "Users can only be invited manually to the organization.",
        manualInvitationOption__label: "No automatic enrollment",
        subtitle: "Choose how users from this domain can join the organization."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Enrollment options"
      },
      subtitle: "The domain {{domain}} is now verified. Continue by selecting enrollment mode.",
      title: "Update {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Enter the verification code sent to your email address",
      formTitle: "Verification code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "The domain {{domainName}} needs to be verified via email.",
      subtitleVerificationCodeScreen: "A verification code was sent to {{emailAddress}}. Enter the code to continue.",
      title: "Verify domain"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "Organizasyon olu\u015Ftur",
    action__invitationAccept: "Join",
    action__manageOrganization: "Organizasyonu y\xF6net",
    action__suggestionsAccept: "Request to join",
    notSelected: "Organizasyon se\xE7ilmedi",
    personalWorkspace: "Ki\u015Fisel \xC7al\u0131\u015Fma Alan\u0131",
    suggestionsAcceptedLabel: "Pending approval"
  },
  paginationButton__next: "\u0130leri",
  paginationButton__previous: "Geri",
  paginationRowText__displaying: "Sayfa bilgisi:",
  paginationRowText__of: "-",
  signIn: {
    accountSwitcher: {
      action__addAccount: "Add account",
      action__signOutAll: "Sign out of all accounts",
      subtitle: "Select the account with which you wish to continue.",
      title: "Choose an account"
    },
    alternativeMethods: {
      actionLink: "Yard\u0131m al",
      actionText: "Don\u2019t have any of these?",
      blockButton__backupCode: "Yedekleme kodu kullan",
      blockButton__emailCode: "{{identifier}} adresine do\u011Frulama kodu g\xF6nder",
      blockButton__emailLink: "{{identifier}} adresine do\u011Frulama ba\u011Flant\u0131s\u0131 g\xF6nder",
      blockButton__passkey: void 0,
      blockButton__password: "\u015Eifreyle giri\u015F yap",
      blockButton__phoneCode: "{{identifier}} numaras\u0131na do\u011Frulama kodu g\xF6nder",
      blockButton__totp: "Authenticator uygulamas\u0131 kullan",
      getHelp: {
        blockButton__emailSupport: "E-posta deste\u011Fi",
        content: "E\u011Fer hesab\u0131n\u0131za giri\u015F yapmakta zorluk ya\u015F\u0131yorsan\u0131z, hesab\u0131n\u0131za eri\u015Fiminizi sa\u011Flayabilmemiz i\xE7in bize bir e-posta g\xF6nderin ve size yard\u0131mc\u0131 olal\u0131m.",
        title: "Yard\u0131m al"
      },
      subtitle: "Facing issues? You can use any of these methods to sign in.",
      title: "Farkl\u0131 bir y\xF6ntem kullan"
    },
    backupCodeMfa: {
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "Yedekleme kodu giri\u015Fi"
    },
    emailCode: {
      formTitle: "Do\u011Frulama kodu",
      resendButton: "Kodu tekrar g\xF6nder",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "E-posta kutunuzu kontrol edin"
    },
    emailLink: {
      expired: {
        subtitle: "Devam etmek i\xE7in en ba\u015Ftaki sekmeye d\xF6n\xFCn",
        title: "Bu do\u011Fru\u015Flama ba\u011Flant\u0131s\u0131n\u0131n s\xFCresi dolmu\u015F"
      },
      failed: {
        subtitle: "Devam etmek i\xE7in en ba\u015Ftaki sekmeye d\xF6n\xFCn",
        title: "Bu do\u011Frulama ba\u011Flant\u0131s\u0131 ge\xE7ersiz"
      },
      formSubtitle: "E-posta adresinize g\xF6nderdi\u011Fimiz do\u011Frulama ba\u011Flant\u0131s\u0131na t\u0131klay\u0131n",
      formTitle: "Do\u011Frulama ba\u011Flant\u0131s\u0131",
      loading: {
        subtitle: "Y\xF6nlediriliyorsunuz...",
        title: "Giri\u015F yap\u0131l\u0131yor..."
      },
      resendButton: "Tekrar g\xF6nder",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "E-posta kutunuzu kontrol edin",
      unusedTab: {
        title: "Bu sekmeyi kapatabilirsiniz"
      },
      verified: {
        subtitle: "Y\xF6nlediriliyorsunuz...",
        title: "Ba\u015Far\u0131yla giri\u015F yap\u0131ld\u0131"
      },
      verifiedSwitchTab: {
        subtitle: "Devam etmek i\xE7in en ba\u015Ftaki sekmeye d\xF6n\xFCn",
        subtitleNewTab: "Devam etmek i\xE7in yeni a\xE7\u0131lm\u0131\u015F sekmeye d\xF6n\xFCn",
        titleNewTab: "Farkl\u0131 bir sekmede giri\u015F yap\u0131ld\u0131"
      }
    },
    forgotPassword: {
      formTitle: "\u015Eifre s\u0131f\u0131rlama kodu",
      resendButton: "Tekrar g\xF6nder",
      subtitle: "to reset your password",
      subtitle_email: "First, enter the code sent to your email ID",
      subtitle_phone: "First, enter the code sent to your phone",
      title: "Reset password"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "\u015Eifremi s\u0131f\u0131rla",
      label__alternativeMethods: "Veya ba\u015Fka bir y\xF6ntem kullan\u0131n:",
      title: "\u015Eifremi unuttum"
    },
    noAvailableMethods: {
      message: "Hesab\u0131n\u0131zda giri\u015F yapmak i\xE7in kullanabilece\u011Finiz bir y\xF6ntem bulunmuyor.",
      subtitle: "Bir hata olu\u015Ftu",
      title: "Giri\u015F yap\u0131lam\u0131yor"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "Ba\u015Fka bir y\xF6ntem kullan",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "\u015Eifrenizi girin"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "Do\u011Frulama kodu",
      resendButton: "Tekrar g\xF6nder",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "Telefonunuza g\xF6z at\u0131n"
    },
    phoneCodeMfa: {
      formTitle: "Do\u011Frulama kodu",
      resendButton: "Tekrar g\xF6nder",
      subtitle: "",
      title: "Telefonunuza g\xF6z at\u0131n"
    },
    resetPassword: {
      formButtonPrimary: "\u015Eifremi s\u0131f\u0131rla",
      requiredMessage: "For security reasons, it is required to reset your password.",
      successMessage: "\u015Eifreniz ba\u015Far\u0131yla s\u0131f\u0131rland\u0131. Oturumunuz a\xE7\u0131l\u0131yor...",
      title: "\u015Eifre s\u0131f\u0131rlama"
    },
    resetPasswordMfa: {
      detailsLabel: "\u015Eifrenizi s\u0131f\u0131rlamadan \xF6nce kimli\u011Finizi do\u011Frulamam\u0131z gerekiyor."
    },
    start: {
      actionLink: "Kay\u0131t ol",
      actionLink__use_email: "E-posta kullan",
      actionLink__use_email_username: "E-posta veya kullan\u0131c\u0131 ad\u0131 kullan",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "Telefon kullan",
      actionLink__use_username: "Kullan\u0131c\u0131 ad\u0131 kullan",
      actionText: "Hesab\u0131n\u0131z yok mu?",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "Giri\u015F yap"
    },
    totpMfa: {
      formTitle: "Do\u011Frulama kodu",
      subtitle: "",
      title: "\u0130ki a\u015Famal\u0131 do\u011Frulama"
    }
  },
  signInEnterPasswordTitle: "\u015Eifrenizi girin",
  signUp: {
    continue: {
      actionLink: "Giri\u015F yap",
      actionText: "Hesab\u0131n\u0131z var m\u0131?",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "Eksik bilgileri tamamlay\u0131n"
    },
    emailCode: {
      formSubtitle: "E-posta adresinize g\xF6nderdi\u011Fimiz do\u011Frulama kodunu giriniz",
      formTitle: "Do\u011Frulama kodu",
      resendButton: "Kodu tekrar g\xF6nder",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "E-posta adresinizi do\u011Frulay\u0131n"
    },
    emailLink: {
      formSubtitle: "E-posta adresinize g\xF6nderdi\u011Fimiz do\u011Frulama ba\u011Flant\u0131s\u0131na t\u0131klay\u0131n",
      formTitle: "Do\u011Frulama ba\u011Flant\u0131s\u0131",
      loading: {
        title: "Giri\u015F yap\u0131l\u0131yor..."
      },
      resendButton: "Tekrar g\xF6nder",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "E-posta adresinizi do\u011Frulay\u0131n",
      verified: {
        title: "Ba\u015Far\u0131yla do\u011Fruland\u0131"
      },
      verifiedSwitchTab: {
        subtitle: "Devam etmek i\xE7in yeni a\xE7\u0131lm\u0131\u015F sekmeye d\xF6n\xFCn",
        subtitleNewTab: "Devam etmek i\xE7in \xF6nceki sekmeye d\xF6n\xFCn",
        title: "E-posta adresiniz ba\u015Far\u0131yla do\u011Fruland\u0131"
      }
    },
    phoneCode: {
      formSubtitle: "Telefon numaran\u0131za g\xF6nderdi\u011Fimiz do\u011Frulama kodunu giriniz",
      formTitle: "Do\u011Frulama kodu",
      resendButton: "Kodu tekrar g\xF6nder",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "Telefon numaran\u0131z\u0131 do\u011Frulay\u0131n"
    },
    start: {
      actionLink: "Giri\u015F yap",
      actionText: "Hesab\u0131n\u0131z var m\u0131?",
      subtitle: "{{applicationName}} ile devam etmek i\xE7in",
      title: "Hesap olu\u015Ftur"
    }
  },
  socialButtonsBlockButton: "{{provider|titleize}} ile giri\u015F yap\u0131n",
  unstable__errors: {
    captcha_invalid: "Sign up unsuccessful due to failed security validations. Please refresh the page to try again or reach out to support for more assistance.",
    captcha_unavailable: "Sign up unsuccessful due to failed bot validation. Please refresh the page to try again or reach out to support for more assistance.",
    form_code_incorrect: "Hatal\u0131 kod",
    form_identifier_exists: "",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "",
    form_param_format_invalid: "",
    form_param_format_invalid__email_address: "Email address must be a valid email address.",
    form_param_format_invalid__phone_number: "Phone number must be in a valid international format",
    form_param_max_length_exceeded__first_name: "First name should not exceed 256 characters.",
    form_param_max_length_exceeded__last_name: "Last name should not exceed 256 characters.",
    form_param_max_length_exceeded__name: "Name should not exceed 256 characters.",
    form_param_nil: "",
    form_password_incorrect: "",
    form_password_length_too_short: "",
    form_password_not_strong_enough: "\u015Eifreniz fazla zay\u0131f",
    form_password_pwned: "Bu \u015Fifre bir veri sald\u0131r\u0131s\u0131nda ele ge\xE7irildi\u011Fi i\xE7in kullan\u0131lamaz. L\xFCtfen ba\u015Fka bir \u015Fifre deneyin.",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "\u015Eifrenize ayr\u0131lan depolama alan\u0131n\u0131 a\u015Ft\u0131n\u0131z. L\xFCtfen daha k\u0131sa bir \u015Fifre deneyin.",
    form_password_validation_failed: "Ge\xE7ersiz \u015Fifre",
    form_username_invalid_character: "",
    form_username_invalid_length: "",
    identification_deletion_failed: "You cannot delete your last identification.",
    not_allowed_access: "",
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "{{length}} karakterden k\u0131sa olmal\u0131",
      minimumLength: "{{length}} veya daha fazla karakter i\xE7ermeli",
      requireLowercase: "bir k\xFC\xE7\xFCk harf i\xE7ermeli",
      requireNumbers: "bir say\u0131 i\xE7ermeli",
      requireSpecialCharacter: "bir \xF6zel karakter i\xE7ermeli",
      requireUppercase: "bir b\xFCy\xFCk harf i\xE7ermeli",
      sentencePrefix: "\u015Eifreniz;"
    },
    phone_number_exists: "This phone number is taken. Please try another.",
    zxcvbn: {
      couldBeStronger: "\u015Eifreniz kriterleri sa\u011Fl\u0131yor, birka\xE7 karakter daha eklerseniz daha g\xFC\xE7l\xFC olacakt\u0131r.",
      goodPassword: "Harika! \u015Eifreniz g\xFC\xE7l\xFC.",
      notEnough: "\u015Eifreniz fazla zay\u0131f",
      suggestions: {
        allUppercase: "T\xFCm harfleri b\xFCy\xFCk yazmak yerine rastgele b\xFCy\xFCk harfler kullan\u0131n.",
        anotherWord: "\u015Eifreniz nadir g\xF6r\xFClen s\xF6zc\xFCklerden daha fazla i\xE7ersin.",
        associatedYears: "Kendinizle alakas\u0131 olan y\u0131llar\u0131 kullanmay\u0131n.",
        capitalization: "Sadece ilk harfi b\xFCy\xFCk yazmak yerine rastgele b\xFCy\xFCk harfler kullan\u0131n.",
        dates: "Kendinizle alakas\u0131 olan tarihleri kullanmay\u0131n.",
        l33t: '"a" yerine "@" kullanmak gibi tahmin edilebilir harf ikamelerinden ka\xE7\u0131n\u0131n.',
        longerKeyboardPattern: "Elinizin y\xF6n\xFCn\xFC birden \xE7ok de\u011Fi\u015Ftirerek yazd\u0131\u011F\u0131n\u0131z uzun klavye desenleri kullan\u0131n.",
        noNeed: "\xD6zel karakter, say\u0131 veya b\xFCy\xFCk harf kullanmadan da g\xFC\xE7l\xFC bir \u015Fifre olu\u015Fturabilirsiniz.",
        pwned: "Bu \u015Fifreyi ba\u015Fka bir yerde kullan\u0131yorsan\u0131z, de\u011Fi\u015Ftirseniz iyi olur.",
        recentYears: "G\xFCn\xFCm\xFCze yak\u0131n y\u0131llar\u0131 kullanmay\u0131n.",
        repeated: "Kelime veya karakterleri tekrarlamay\u0131n.",
        reverseWords: "Yayg\u0131n kelimelerin tersini kullanmaktan ka\xE7\u0131n\u0131n.",
        sequences: "Yayg\u0131n desenleri kullanmay\u0131n.",
        useWords: "Birden \xE7ok s\xF6zc\xFCk kullan\u0131n ama yayg\u0131n deyi\u015Flerden ka\xE7\u0131n\u0131n."
      },
      warnings: {
        common: "Bu yayg\u0131n bir \u015Fifre.",
        commonNames: "Yayg\u0131n adlar ve soyadlar kolay tahmin edilir.",
        dates: "Tarihleri tahmin etmek kolayd\u0131r.",
        extendedRepeat: '"abcabcabc" gibi tekrarlanan desenler kolay tahmin edilir.',
        keyPattern: "K\u0131sa klavye desenleri kolay tahmin edilir.",
        namesByThemselves: "Adlar ya da soyadlar kolay tahmin edilir.",
        pwned: "Bu \u015Fifre bir veri sald\u0131r\u0131s\u0131nda ele ge\xE7irilmi\u015F.",
        recentYears: "Yak\u0131n y\u0131llar kolay tahmin edilir.",
        sequences: '"abc" gibi yayg\u0131n desenlerin tahmini kolayd\u0131r.',
        similarToCommon: "Bu yayg\u0131n kullan\u0131lan \u015Fifrelere \xE7ok benziyor.",
        simpleRepeat: "Tekrarlanan karakterler kolay tahmin edilir.",
        straightRow: "Klavyede ard\u0131\u015F\u0131k karakterler kolay tahmin edilir.",
        topHundred: "Bu \xE7ok yayg\u0131n bir \u015Fifre.",
        topTen: "Bu epey yayg\u0131n bir \u015Fifre.",
        userInputs: "\u015Eifreniz ki\u015Fisel bilgiler i\xE7ermemeli.",
        wordByItself: "Tek s\xF6zc\xFCkler kolay tahmin edilir."
      }
    }
  },
  userButton: {
    action__addAccount: "Hesap ekle",
    action__manageAccount: "Hesab\u0131 y\xF6net",
    action__signOut: "\xC7\u0131k\u0131\u015F yap",
    action__signOutAll: "T\xFCm hesaplardan \xE7\u0131k\u0131\u015F yap"
  },
  userProfile: {
    backupCodePage: {
      actionLabel__copied: "Kopyaland\u0131!",
      actionLabel__copy: "Hepsini kopayala",
      actionLabel__download: ".txt olarak indir",
      actionLabel__print: "Yazd\u0131r",
      infoText1: "Yedekleme kodlar\u0131 bu hesap i\xE7in etkinle\u015Ftirilecektir.",
      infoText2: "Yedekleme kodlar\u0131n\u0131z\u0131 g\xFCvenli bir yerde saklay\u0131n. E\u011Fer bu kodlar\u0131n\u0131z\u0131n ba\u015Fkas\u0131n\u0131n eline ge\xE7ti\u011Fini d\xFC\u015F\xFCn\xFCrseniz, yenilerini olu\u015Fturabilirsiniz.",
      subtitle__codelist: "Yedekleme kodlar\u0131n\u0131z\u0131 g\xFCvenli bir yerde saklay\u0131n.",
      successMessage: "Yedekleme kodlar\u0131 ba\u015Far\u0131yla eklendi. E\u011Fer Authenticator uygulaman\u0131z\u0131n oldu\u011Fu cihaza eri\u015Fiminizi kaybettiyseniz, oturum a\xE7arken bu kodlardan birini girebilirsiniz. Her kod en fazla bir kez kullan\u0131labilir.",
      successSubtitle: "E\u011Fer Authenticator uygulaman\u0131z\u0131n oldu\u011Fu cihaz\u0131n\u0131za eri\u015Fiminizi kaybederseniz, bu kodlardan birini kullanarak hesab\u0131n\u0131za giri\u015F yapabilirsiniz.",
      title: "Yedekleme kodu do\u011Frulamas\u0131 ekle",
      title__codelist: "Yedekleme kodlar\u0131"
    },
    connectedAccountPage: {
      formHint: "Yeni bir hesap ba\u011Flamak i\xE7in bir sa\u011Flay\u0131c\u0131 se\xE7iniz.",
      formHint__noAccounts: "Kullan\u0131labilir bir sa\u011Flay\u0131c\u0131 yok.",
      removeResource: {
        messageLine1: "{{identifier}} hesab\u0131n\u0131zdan kald\u0131r\u0131lacakt\u0131r.",
        messageLine2: "Art\u0131k bu ba\u011Fl\u0131 hesab\u0131 kullanarak oturum a\xE7man\u0131z m\xFCmk\xFCn olmayacakt\u0131r ve buna ba\u011Fl\u0131 \xF6zellikler \xE7al\u0131\u015Fmayacakt\u0131r.",
        successMessage: "{{connectedAccount}} hesab\u0131n\u0131zdan kald\u0131r\u0131ld\u0131.",
        title: "Ba\u011Fl\u0131 hesab\u0131 kald\u0131r"
      },
      socialButtonsBlockButton: "{{provider|titleize}} hesab\u0131 ba\u011Fla",
      successMessage: "Sa\u011Flay\u0131c\u0131 hesab\u0131n\u0131za ba\u011Fland\u0131.",
      title: "Hesap ba\u011Fla"
    },
    deletePage: {
      actionDescription: 'Type "Delete account" below to continue.',
      confirm: "Delete account",
      messageLine1: "Are you sure you want to delete your account?",
      messageLine2: "This action is permanent and irreversible.",
      title: "Delete account"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "Do\u011Frulama kodunu i\xE7eren bir e-posta belirtti\u011Finiz adrese g\xF6nderilecektir.",
        formSubtitle: "{{identifier}} adresine g\xF6nderilen do\u011Frulama kodunu giriniz",
        formTitle: "Do\u011Frulama kodu",
        resendButton: "Yeniden g\xF6nder",
        successMessage: "{{identifier}} adresi hesab\u0131n\u0131za eklendi."
      },
      emailLink: {
        formHint: "Do\u011Frulama ba\u011Flant\u0131s\u0131n\u0131 i\xE7eren bir e-posta belirtti\u011Finiz adrese g\xF6nderilecektir.",
        formSubtitle: "{{identifier}} adresine g\xF6nderilen do\u011Frulama ba\u011Flant\u0131s\u0131n\u0131 t\u0131klay\u0131n\u0131z",
        formTitle: "Do\u011Frulama ba\u011Flant\u0131s\u0131",
        resendButton: "Yeniden g\xF6nder",
        successMessage: "{{identifier}} adresi hesab\u0131n\u0131za eklendi."
      },
      removeResource: {
        messageLine1: "{{identifier}} adresi hesab\u0131n\u0131zdan kald\u0131r\u0131lacakt\u0131r.",
        messageLine2: "Art\u0131k bu e-posta adresini kullanarak oturum a\xE7man\u0131z m\xFCmk\xFCn olmayacakt\u0131r.",
        successMessage: "{{emailAddress}} adresi hesab\u0131n\u0131zdan kald\u0131r\u0131ld\u0131.",
        title: "E-posta adresini kald\u0131r"
      },
      title: "E-posta adresi ekle",
      verifyTitle: "Verify email address"
    },
    formButtonPrimary__add: "Add",
    formButtonPrimary__continue: "\u0130lerle",
    formButtonPrimary__finish: "Bitir",
    formButtonPrimary__remove: "Remove",
    formButtonPrimary__save: "Save",
    formButtonReset: "\u0130ptal",
    mfaPage: {
      formHint: "Eklemek i\xE7in bir y\xF6ntem se\xE7iniz.",
      title: "\u0130ki a\u015Famal\u0131 do\u011Frulama y\xF6ntemi ekle"
    },
    mfaPhoneCodePage: {
      backButton: "Use existing number",
      primaryButton__addPhoneNumber: "Telefon numaras\u0131 ekle",
      removeResource: {
        messageLine1: "Giri\u015F yaparken art\u0131k {{identifier}} numaras\u0131na SMS kodu g\xF6nderilmeyecektir.",
        messageLine2: "Hesab\u0131n\u0131z\u0131n g\xFCvenli\u011Fi azalabilir. Devam etmek istedi\u011Finizden emin misiniz?",
        successMessage: "\u0130ki a\u015Famal\u0131 SMS kodu do\u011Frulamas\u0131 {{mfaPhoneCode}} numaras\u0131ndan kald\u0131r\u0131ld\u0131.",
        title: "\u0130ki a\u015Famal\u0131 do\u011Frulamay\u0131 kald\u0131r"
      },
      subtitle__availablePhoneNumbers: "\u0130ki a\u015Famal\u0131 SMS kodu do\u011Frulamas\u0131 i\xE7in bir telefon numaras\u0131 se\xE7in.",
      subtitle__unavailablePhoneNumbers: "\u0130ki a\u015Famal\u0131 SMS kodu do\u011Frulamas\u0131 i\xE7in kullan\u0131labilir bir telefon numaras\u0131 yok.",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS code verification enabled",
      title: "SMS kodu do\u011Frulamas\u0131 ekle"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Veya QR kodunu tarat",
        buttonUnableToScan__nonPrimary: "QR kodunu tarayam\u0131yorum",
        infoText__ableToScan: "Authenticator uygulaman\u0131zda yeni bir giri\u015F y\xF6ntemi ayarlay\u0131n ve hesab\u0131n\u0131zla ba\u011Flamak i\xE7in a\u015Fa\u011F\u0131daki QR kodunu taray\u0131n.",
        infoText__unableToScan: "Authenticator uygulaman\u0131zda yeni bir giri\u015F y\xF6ntemi ekleme se\xE7ene\u011Fini bulun ve a\u015Fa\u011F\u0131da verilen de\u011Feri girin:",
        inputLabel__unableToScan1: "Zaman bazl\u0131 veya tek seferlik \u015Fifrelerin etkinle\u015Ftirildi\u011Finden emin olun, ard\u0131ndan hesab\u0131n\u0131z\u0131 ba\u011Flamay\u0131 tamamlay\u0131n.",
        inputLabel__unableToScan2: "Alternatif olarak do\u011Frulay\u0131c\u0131n\u0131z TOTP URI\u2019leri destekliyorsa, tam URI\u2019yi de kopyalayabilirsiniz."
      },
      removeResource: {
        messageLine1: "Art\u0131k giri\u015F yaparken authenticator'dan gelecek do\u011Frulama kodlar\u0131 gerekmeyecektir.",
        messageLine2: "Hesab\u0131n\u0131z\u0131n g\xFCvenli\u011Fi azalabilir. Devam etmek istedi\u011Finizden emin misiniz?",
        successMessage: "\u0130ki a\u015Famal\u0131 do\u011Frulama y\xF6ntemi kald\u0131r\u0131ld\u0131.",
        title: "\u0130ki a\u015Famal\u0131 do\u011Frulamay\u0131 kald\u0131r"
      },
      successMessage: "\u0130ki a\u015Famal\u0131 do\u011Frulama y\xF6ntemi ba\u015Far\u0131yla eklendi. Oturum a\xE7arken, ek bir ad\u0131m olarak bu do\u011Frulay\u0131c\u0131dan bir do\u011Frulama kodu girmeniz gerekecektir.",
      title: "Authenticator uygulamas\u0131 ekle",
      verifySubtitle: "Authenticator uygulaman\u0131zda olu\u015Fturulan do\u011Frulama kodunu giriniz",
      verifyTitle: "Do\u011Frulama kodu"
    },
    mobileButton__menu: "Men\xFC",
    navbar: {
      account: "Profile",
      description: "Manage your account info.",
      security: "Security",
      title: "Account"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "It is recommended to sign out of all other devices which may have used your old password.",
      readonly: "Your password can currently not be edited because you can sign in only via the enterprise connection.",
      successMessage__set: "\u015Eifreniz ba\u015Far\u0131yla de\u011Fi\u015Ftirildi.",
      successMessage__signOutOfOtherSessions: "Di\u011Fer t\xFCm cihazlardaki oturumlar\u0131n\u0131z sonland\u0131r\u0131ld\u0131.",
      successMessage__update: "\u015Eifreniz g\xFCnceellendi.",
      title__set: "\u015Eifreyi de\u011Fi\u015Ftir",
      title__update: "Yeni \u015Fifre girin"
    },
    phoneNumberPage: {
      infoText: "Belirtilen numaraya do\u011Frulama kodunu i\xE7eren bir SMS g\xF6nderilecektir.",
      removeResource: {
        messageLine1: "{{identifier}} numaras\u0131 hesab\u0131n\u0131zdan kald\u0131r\u0131lacakt\u0131r.",
        messageLine2: "Art\u0131k bu telefon numaras\u0131n\u0131 kullanarak oturum a\xE7man\u0131z m\xFCmk\xFCn olmayacakt\u0131r.",
        successMessage: "{{phoneNumber}} numaras\u0131 hesab\u0131n\u0131zdan kald\u0131r\u0131ld\u0131.",
        title: "Telefon numaras\u0131n\u0131 kald\u0131r"
      },
      successMessage: "{{identifier}} numaras\u0131 hesab\u0131n\u0131za eklendi.",
      title: "Telefon numaras\u0131 ekle",
      verifySubtitle: "Enter the verification code sent to {{identifier}}",
      verifyTitle: "Verify phone number"
    },
    profilePage: {
      fileDropAreaHint: "10 MB'tan k\xFC\xE7\xFCk boyutta bir JPG, PNG, GIF, veya WEBP dosyas\u0131 y\xFCkle",
      imageFormDestructiveActionSubtitle: "G\xF6rseli kald\u0131r",
      imageFormSubtitle: "G\xF6rsel y\xFCkle",
      imageFormTitle: "Profil g\xF6rseli",
      readonly: "Your profile information has been provided by the enterprise connection and cannot be edited.",
      successMessage: "Profiliniz g\xFCncellendi.",
      title: "Profili g\xFCncelle"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Cihaz oturumunu sonland\u0131r",
        title: "Aktif cihazlar"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "Yeniden dene",
        actionLabel__reauthorize: "Yetkilendir",
        destructiveActionTitle: "Kald\u0131r",
        primaryButton: "Hesap ba\u011Fla",
        subtitle__reauthorize: "The required scopes have been updated, and you may be experiencing limited functionality. Please re-authorize this application to avoid any issues",
        title: "Ba\u011Fl\u0131 hesaplar"
      },
      dangerSection: {
        deleteAccountButton: "Hesab\u0131 sil",
        title: "Tehlike"
      },
      emailAddressesSection: {
        destructiveAction: "E-posta adresini kald\u0131r",
        detailsAction__nonPrimary: "Birincil e-posta adresi yap",
        detailsAction__primary: "Do\u011Frulamay\u0131 tamamla",
        detailsAction__unverified: "Do\u011Frulamay\u0131 tamamla",
        primaryButton: "E-posta adresi ekle",
        title: "E-posta adresleri"
      },
      enterpriseAccountsSection: {
        title: "Enterprise accounts"
      },
      headerTitle__account: "Hesap",
      headerTitle__security: "G\xFCvenlik",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "Kodlar\u0131 yenile",
          headerTitle: "Yedekleme kodlar\u0131",
          subtitle__regenerate: "Yeni bir dizi g\xFCvenli yedekleme kodu al\u0131n. \xD6nceki yedekleme kodlar\u0131 silinecek ve kullan\u0131lamayacakt\u0131r.",
          title__regenerate: "Yedekleme kodlar\u0131n\u0131 yenile"
        },
        phoneCode: {
          actionLabel__setDefault: "Varsay\u0131lan olarak ayarla",
          destructiveActionLabel: "Telefon numaras\u0131n\u0131 kald\u0131r"
        },
        primaryButton: "\u0130ki a\u015Famal\u0131 do\u011Frulama ekle",
        title: "\u0130ki a\u015Famal\u0131 do\u011Frulama",
        totp: {
          destructiveActionTitle: "Kald\u0131r",
          headerTitle: "Authenticator uygulamas\u0131"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "\u015Eifreyi g\xFCncelle",
        primaryButton__updatePassword: "\u015Eifreyi de\u011Fi\u015Ftir",
        title: "\u015Eifre"
      },
      phoneNumbersSection: {
        destructiveAction: "Telefon numaras\u0131n\u0131 kald\u0131r",
        detailsAction__nonPrimary: "Birincil yap",
        detailsAction__primary: "Do\u011Frulamay\u0131 tamamla",
        detailsAction__unverified: "Do\u011Frulamay\u0131 tamamla",
        primaryButton: "Telefon numaras\u0131 ekle",
        title: "Telefon numaralar\u0131"
      },
      profileSection: {
        primaryButton: "",
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "G\xFCncelle",
        primaryButton__updateUsername: "Kullan\u0131c\u0131 ad\u0131n\u0131 de\u011Fi\u015Ftir",
        title: "Kullan\u0131c\u0131 ad\u0131"
      },
      web3WalletsSection: {
        destructiveAction: "C\xFCzdan\u0131 kald\u0131r",
        primaryButton: "Web3 c\xFCzdanlar\u0131",
        title: "Web3 c\xFCzdanlar\u0131"
      }
    },
    usernamePage: {
      successMessage: "Kullan\u0131c\u0131 ad\u0131n\u0131z g\xFCncellendi.",
      title__set: "Kullan\u0131c\u0131 ad\u0131n\u0131 g\xFCncelle",
      title__update: "Kullan\u0131c\u0131 ad\u0131n\u0131 g\xFCncelle"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} c\xFCzdan\u0131 hesab\u0131n\u0131zdan kald\u0131r\u0131lacakt\u0131r.",
        messageLine2: "Art\u0131k bu c\xFCzdan\u0131 kullanarak oturum a\xE7man\u0131z m\xFCmk\xFCn olmayacakt\u0131r.",
        successMessage: "{{web3Wallet}} c\xFCzdan\u0131 hesab\u0131n\u0131zdan kald\u0131r\u0131ld\u0131.",
        title: "Web3 c\xFCzdan\u0131n\u0131 kald\u0131r"
      },
      subtitle__availableWallets: "Hesab\u0131n\u0131za eklemek i\xE7in bir web3 c\xFCzdan\u0131 se\xE7iniz.",
      subtitle__unavailableWallets: "Kullan\u0131labilir bir web3 c\xFCzdan\u0131 yok.",
      successMessage: "Web3 c\xFCzdan\u0131n\u0131z hesab\u0131n\u0131za eklendi.",
      title: "Web3 c\xFCzdan\u0131 ekle"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  trTR
});
